// @rc-component/dialog/dist/common-async.4704580574928883170.hot-update.js
globalThis.makoModuleHotUpdate('common', {
    modules: {
        "src/Dialog/Content/index.tsx": function(module, exports, __mako_require__) {
            "use strict";
            __mako_require__.d(exports, "__esModule", {
                value: true
            });
            __mako_require__.d(exports, "default", {
                enumerable: true,
                get: function() {
                    return _default;
                }
            });
            var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
            var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
            var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
            var _jsxdevruntime = __mako_require__("node_modules/_react@18.3.1@react/jsx-dev-runtime.js");
            var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react@18.3.1@react/index.js"));
            var _classnames = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_classnames@2.5.1@classnames/index.js"));
            var _motion = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_@rc-component_motion@1.1.3@@rc-component/motion/es/index.js"));
            var _util = __mako_require__("src/util.ts");
            var _Panel = /*#__PURE__*/ _interop_require_default._(__mako_require__("src/Dialog/Content/Panel.tsx"));
            var prevRefreshReg;
            var prevRefreshSig;
            prevRefreshReg = self.$RefreshReg$;
            prevRefreshSig = self.$RefreshSig$;
            self.$RefreshReg$ = (type, id)=>{
                _reactrefresh.register(type, module.id + id);
            };
            self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
            var _s = $RefreshSig$();
            const Content = /*#__PURE__*/ _s(_react.forwardRef(_c = _s((props, ref)=>{
                _s();
                const { prefixCls, title, style, className, visible, forceRender, destroyOnClose, motionName, ariaId, onVisibleChanged, mousePosition } = props;
                const dialogRef = (0, _react.useRef)();
                const panelRef = (0, _react.useRef)();
                // ============================== Refs ==============================
                _react.useImperativeHandle(ref, ()=>({
                        ...panelRef.current,
                        inMotion: dialogRef.current.inMotion,
                        enableMotion: dialogRef.current.enableMotion
                    }));
                // ============================= Style ==============================
                const [transformOrigin, setTransformOrigin] = _react.useState();
                const contentStyle = {};
                if (transformOrigin) contentStyle.transformOrigin = transformOrigin;
                function onPrepare() {
                    const elementOffset = (0, _util.offset)(dialogRef.current.nativeElement);
                    setTransformOrigin(mousePosition && (mousePosition.x || mousePosition.y) ? `${mousePosition.x - elementOffset.left}px ${mousePosition.y - elementOffset.top}px` : '');
                }
                // ============================= Render =============================
                return /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_motion.default, {
                    visible: visible,
                    onVisibleChanged: onVisibleChanged,
                    onAppearPrepare: onPrepare,
                    onEnterPrepare: onPrepare,
                    forceRender: forceRender,
                    motionName: motionName,
                    removeOnLeave: destroyOnClose,
                    ref: dialogRef,
                    children: ({ className: motionClassName, style: motionStyle }, motionRef)=>/*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_Panel.default, {
                            ...props,
                            ref: panelRef,
                            title: title,
                            ariaId: ariaId,
                            prefixCls: prefixCls,
                            holderRef: motionRef,
                            style: {
                                ...motionStyle,
                                ...style,
                                ...contentStyle
                            },
                            className: (0, _classnames.default)(className, motionClassName)
                        }, void 0, false, {
                            fileName: "src/Dialog/Content/index.tsx",
                            lineNumber: 77,
                            columnNumber: 9
                        }, this)
                }, void 0, false, {
                    fileName: "src/Dialog/Content/index.tsx",
                    lineNumber: 66,
                    columnNumber: 5
                }, this);
            }, "B8iJBl3QRco45KFPnPJ2XBRLfiA=")), "B8iJBl3QRco45KFPnPJ2XBRLfiA=");
            _c1 = Content;
            Content.displayName = 'Content';
            var _default = Content;
            var _c;
            var _c1;
            $RefreshReg$(_c, "Content$React.forwardRef");
            $RefreshReg$(_c1, "Content");
            if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
            if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
            function registerClassComponent(filename, moduleExports) {
                for(const key in moduleExports)try {
                    if (key === "__esModule") continue;
                    const exportValue = moduleExports[key];
                    if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
                } catch (e) {}
            }
            function $RefreshIsReactComponentLike$(moduleExports) {
                if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
                for(var key in moduleExports)try {
                    if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
                } catch (e) {}
                return false;
            }
            registerClassComponent(module.id, module.exports);
            if ($RefreshIsReactComponentLike$(module.exports)) {
                module.meta.hot.accept();
                _reactrefresh.performReactRefresh();
            }
        }
    }
}, function(runtime) {
    runtime._h = '960384830459946830';
    runtime.updateEnsure2Map({
        ".dumi/tmp/dumi/theme/ContextWrapper.tsx": [
            "dumi__tmp__dumi__theme__ContextWrapper"
        ],
        "docs/changelog.md": [
            "vendors",
            "docs/changelog.md"
        ],
        "docs/changelog.md?type=demo": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/changelog.md?type=text": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/demo/ant-design.md": [
            "vendors",
            "docs/demo/ant-design.md"
        ],
        "docs/demo/ant-design.md?type=demo": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/demo/ant-design.md?type=text": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/demo/bootstrap.md": [
            "vendors",
            "docs/demo/bootstrap.md"
        ],
        "docs/demo/bootstrap.md?type=demo": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/demo/bootstrap.md?type=text": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/demo/draggable.md": [
            "vendors",
            "docs/demo/draggable.md"
        ],
        "docs/demo/draggable.md?type=demo": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/demo/draggable.md?type=text": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/demo/multiple-Portal.md": [
            "vendors",
            "docs/demo/multiple-Portal.md"
        ],
        "docs/demo/multiple-Portal.md?type=demo": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/demo/multiple-Portal.md?type=text": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/demo/pure.md": [
            "vendors",
            "docs/demo/pure.md"
        ],
        "docs/demo/pure.md?type=demo": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/demo/pure.md?type=text": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/examples/ant-design.tsx?techStack=react": [
            "vendors",
            "common",
            "demos"
        ],
        "docs/examples/bootstrap.tsx?techStack=react": [
            "vendors",
            "common",
            "demos"
        ],
        "docs/examples/draggable.tsx?techStack=react": [
            "vendors",
            "common",
            "demos"
        ],
        "docs/examples/multiple-Portal.tsx?techStack=react": [
            "vendors",
            "common",
            "demos"
        ],
        "docs/examples/pure.tsx?techStack=react": [
            "vendors",
            "common",
            "demos"
        ],
        "docs/index.md": [
            "vendors",
            "docs/index.md"
        ],
        "docs/index.md?type=demo": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/index.md?type=text": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "node_modules/_dumi@2.4.17@dumi/dist/client/misc/reactDemoCompiler.js": [
            "vendors",
            "node_modules/_dumi@2.4.17@dumi/dist/client/misc/reactDemoCompiler.js"
        ],
        "node_modules/_dumi@2.4.17@dumi/dist/client/pages/404.js": [
            "vendors",
            "nm__dumi__dist__client__pages__404"
        ],
        "node_modules/_dumi@2.4.17@dumi/dist/client/pages/Demo/index.js": [
            "vendors",
            "nm__dumi__dist__client__pages__Demo__index"
        ],
        "node_modules/_dumi@2.4.17@dumi/theme-default/layouts/DocLayout/index.js": [
            "vendors",
            "node_modules/_dumi@2.4.17@dumi/theme-default/layouts/DocLayout/index.js"
        ],
        "node_modules/_react-dom@18.3.1@react-dom/server.browser.js": [
            "vendors",
            "node_modules/_react-dom@18.3.1@react-dom/server.browser.js"
        ]
    });
    ;
});
