// @rc-component/dialog/dist/common-async.js
((typeof globalThis !== 'undefined' ? globalThis : self)["makoChunk_@rc-component/dialog"] = (typeof globalThis !== 'undefined' ? globalThis : self)["makoChunk_@rc-component/dialog"] || []).push([
        ['common'],
{ "docs/examples/ant-design.tsx?watch=parent": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/examples/bootstrap.tsx?watch=parent": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/examples/draggable.tsx?watch=parent": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/examples/multiple-Portal.tsx?watch=parent": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/examples/pure.tsx?watch=parent": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"src/Dialog/Content/MemoChildren.tsx": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react@18.3.1@react/index.js"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _default = /*#__PURE__*/ _c1 = _react.memo(_c = ({ children })=>children, (_, { shouldUpdate })=>!shouldUpdate);
var _c;
var _c1;
$RefreshReg$(_c, "%default%$React.memo");
$RefreshReg$(_c1, "%default%");
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"src/Dialog/Content/Panel.tsx": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
var _jsxdevruntime = __mako_require__("node_modules/_react@18.3.1@react/jsx-dev-runtime.js");
var _classnames = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_classnames@2.5.1@classnames/index.js"));
var _ref = __mako_require__("node_modules/_@rc-component_util@1.2.1@@rc-component/util/lib/ref.js");
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react@18.3.1@react/index.js"));
var _context = __mako_require__("src/context.ts");
var _MemoChildren = /*#__PURE__*/ _interop_require_default._(__mako_require__("src/Dialog/Content/MemoChildren.tsx"));
var _pickAttrs = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_@rc-component_util@1.2.1@@rc-component/util/lib/pickAttrs.js"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _s = $RefreshSig$();
const sentinelStyle = {
    width: 0,
    height: 0,
    overflow: 'hidden',
    outline: 'none'
};
const entityStyle = {
    outline: 'none'
};
const Panel = /*#__PURE__*/ _s(_react.default.forwardRef(_c = _s((props, ref)=>{
    _s();
    const { prefixCls, className, style, title, ariaId, footer, closable, closeIcon, onClose, children, bodyStyle, bodyProps, modalRender, onMouseDown, onMouseUp, holderRef, visible, forceRender, width, height, classNames: modalClassNames, styles: modalStyles } = props;
    // ================================= Refs =================================
    const { panel: panelRef } = _react.default.useContext(_context.RefContext);
    const mergedRef = (0, _ref.useComposeRef)(holderRef, panelRef);
    const sentinelStartRef = (0, _react.useRef)();
    const sentinelEndRef = (0, _react.useRef)();
    _react.default.useImperativeHandle(ref, ()=>({
            focus: ()=>{
                var _sentinelStartRef_current;
                (_sentinelStartRef_current = sentinelStartRef.current) === null || _sentinelStartRef_current === void 0 || _sentinelStartRef_current.focus({
                    preventScroll: true
                });
            },
            changeActive: (next)=>{
                const { activeElement } = document;
                if (next && activeElement === sentinelEndRef.current) sentinelStartRef.current.focus({
                    preventScroll: true
                });
                else if (!next && activeElement === sentinelStartRef.current) sentinelEndRef.current.focus({
                    preventScroll: true
                });
            }
        }));
    // ================================ Style =================================
    const contentStyle = {};
    if (width !== undefined) contentStyle.width = width;
    if (height !== undefined) contentStyle.height = height;
    // ================================ Render ================================
    const footerNode = footer ? /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
        className: (0, _classnames.default)(`${prefixCls}-footer`, modalClassNames === null || modalClassNames === void 0 ? void 0 : modalClassNames.footer),
        style: {
            ...modalStyles === null || modalStyles === void 0 ? void 0 : modalStyles.footer
        },
        children: footer
    }, void 0, false, {
        fileName: "src/Dialog/Content/Panel.tsx",
        lineNumber: 84,
        columnNumber: 5
    }, this) : null;
    const headerNode = title ? /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
        className: (0, _classnames.default)(`${prefixCls}-header`, modalClassNames === null || modalClassNames === void 0 ? void 0 : modalClassNames.header),
        style: {
            ...modalStyles === null || modalStyles === void 0 ? void 0 : modalStyles.header
        },
        children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
            className: (0, _classnames.default)(`${prefixCls}-title`, modalClassNames === null || modalClassNames === void 0 ? void 0 : modalClassNames.title),
            id: ariaId,
            style: {
                ...modalStyles === null || modalStyles === void 0 ? void 0 : modalStyles.title
            },
            children: title
        }, void 0, false, {
            fileName: "src/Dialog/Content/Panel.tsx",
            lineNumber: 97,
            columnNumber: 7
        }, this)
    }, void 0, false, {
        fileName: "src/Dialog/Content/Panel.tsx",
        lineNumber: 93,
        columnNumber: 5
    }, this) : null;
    const closableObj = (0, _react.useMemo)(()=>{
        if (typeof closable === 'object' && closable !== null) return closable;
        if (closable) return {
            closeIcon: closeIcon ?? /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                className: `${prefixCls}-close-x`
            }, void 0, false, {
                fileName: "src/Dialog/Content/Panel.tsx",
                lineNumber: 112,
                columnNumber: 40
            }, this)
        };
        return {};
    }, [
        closable,
        closeIcon,
        prefixCls
    ]);
    const ariaProps = (0, _pickAttrs.default)(closableObj, true);
    const closeBtnIsDisabled = typeof closable === 'object' && closable.disabled;
    const closerNode = closable ? /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("button", {
        type: "button",
        onClick: onClose,
        "aria-label": "Close",
        ...ariaProps,
        className: `${prefixCls}-close`,
        disabled: closeBtnIsDisabled,
        children: closableObj.closeIcon
    }, void 0, false, {
        fileName: "src/Dialog/Content/Panel.tsx",
        lineNumber: 121,
        columnNumber: 5
    }, this) : null;
    const content = /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
        className: (0, _classnames.default)(`${prefixCls}-section`, modalClassNames === null || modalClassNames === void 0 ? void 0 : modalClassNames.section),
        style: modalStyles === null || modalStyles === void 0 ? void 0 : modalStyles.section,
        children: [
            closerNode,
            headerNode,
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
                className: (0, _classnames.default)(`${prefixCls}-body`, modalClassNames === null || modalClassNames === void 0 ? void 0 : modalClassNames.body),
                style: {
                    ...bodyStyle,
                    ...modalStyles === null || modalStyles === void 0 ? void 0 : modalStyles.body
                },
                ...bodyProps,
                children: children
            }, void 0, false, {
                fileName: "src/Dialog/Content/Panel.tsx",
                lineNumber: 140,
                columnNumber: 7
            }, this),
            footerNode
        ]
    }, void 0, true, {
        fileName: "src/Dialog/Content/Panel.tsx",
        lineNumber: 134,
        columnNumber: 5
    }, this);
    return /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
        role: "dialog",
        "aria-labelledby": title ? ariaId : null,
        "aria-modal": "true",
        ref: mergedRef,
        style: {
            ...style,
            ...contentStyle
        },
        className: (0, _classnames.default)(prefixCls, className),
        onMouseDown: onMouseDown,
        onMouseUp: onMouseUp,
        children: [
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
                ref: sentinelStartRef,
                tabIndex: 0,
                style: entityStyle,
                children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_MemoChildren.default, {
                    shouldUpdate: visible || forceRender,
                    children: modalRender ? modalRender(content) : content
                }, void 0, false, {
                    fileName: "src/Dialog/Content/Panel.tsx",
                    lineNumber: 164,
                    columnNumber: 9
                }, this)
            }, void 0, false, {
                fileName: "src/Dialog/Content/Panel.tsx",
                lineNumber: 163,
                columnNumber: 7
            }, this),
            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
                tabIndex: 0,
                ref: sentinelEndRef,
                style: sentinelStyle
            }, void 0, false, {
                fileName: "src/Dialog/Content/Panel.tsx",
                lineNumber: 168,
                columnNumber: 7
            }, this)
        ]
    }, "dialog-element", true, {
        fileName: "src/Dialog/Content/Panel.tsx",
        lineNumber: 152,
        columnNumber: 5
    }, this);
}, "WDebUpzP8NNHE6ypquv/UVUG6v0=", false, function() {
    return [
        _ref.useComposeRef
    ];
})), "WDebUpzP8NNHE6ypquv/UVUG6v0=", false, function() {
    return [
        _ref.useComposeRef
    ];
});
_c1 = Panel;
Panel.displayName = 'Panel';
var _default = Panel;
var _c;
var _c1;
$RefreshReg$(_c, "Panel$React.forwardRef");
$RefreshReg$(_c1, "Panel");
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"src/Dialog/Content/index.tsx": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
var _jsxdevruntime = __mako_require__("node_modules/_react@18.3.1@react/jsx-dev-runtime.js");
var _react = _interop_require_wildcard._(__mako_require__("node_modules/_react@18.3.1@react/index.js"));
var _classnames = _interop_require_default._(__mako_require__("node_modules/_classnames@2.5.1@classnames/index.js"));
var _motion = _interop_require_default._(__mako_require__("node_modules/_@rc-component_motion@1.1.3@@rc-component/motion/es/index.js"));
var _util = __mako_require__("src/util.ts");
var _Panel = _interop_require_default._(__mako_require__("src/Dialog/Content/Panel.tsx"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _s = $RefreshSig$();
const Content = _s(_react.forwardRef(_c = _s((props, ref)=>{
    _s();
    const { prefixCls, title, style, className, visible, forceRender, destroyOnClose, motionName, ariaId, onVisibleChanged, mousePosition } = props;
    const dialogRef = (0, _react.useRef)();
    const panelRef = (0, _react.useRef)();
    _react.useImperativeHandle(ref, ()=>({
            ...panelRef.current,
            inMotion: dialogRef.current.inMotion,
            enableMotion: dialogRef.current.enableMotion
        }));
    const [transformOrigin, setTransformOrigin] = _react.useState();
    const contentStyle = {};
    if (transformOrigin) contentStyle.transformOrigin = transformOrigin;
    function onPrepare() {
        const elementOffset = (0, _util.offset)(dialogRef.current.nativeElement);
        setTransformOrigin(mousePosition && (mousePosition.x || mousePosition.y) ? `${mousePosition.x - elementOffset.left}px ${mousePosition.y - elementOffset.top}px` : '');
    }
    return (0, _jsxdevruntime.jsxDEV)(_motion.default, {
        visible: visible,
        onVisibleChanged: onVisibleChanged,
        onAppearPrepare: onPrepare,
        onEnterPrepare: onPrepare,
        forceRender: forceRender,
        motionName: motionName,
        removeOnLeave: destroyOnClose,
        ref: dialogRef,
        children: ({ className: motionClassName, style: motionStyle }, motionRef)=>(0, _jsxdevruntime.jsxDEV)(_Panel.default, {
                ...props,
                ref: panelRef,
                title: title,
                ariaId: ariaId,
                prefixCls: prefixCls,
                holderRef: motionRef,
                style: {
                    ...motionStyle,
                    ...style,
                    ...contentStyle
                },
                className: (0, _classnames.default)(className, motionClassName)
            }, void 0, false, {
                fileName: "src/Dialog/Content/index.tsx",
                lineNumber: 81,
                columnNumber: 9
            }, this)
    }, void 0, false, {
        fileName: "src/Dialog/Content/index.tsx",
        lineNumber: 70,
        columnNumber: 5
    }, this);
}, "B8iJBl3QRco45KFPnPJ2XBRLfiA=")), "B8iJBl3QRco45KFPnPJ2XBRLfiA=");
_c1 = Content;
Content.displayName = 'Content';
var _default = Content;
var _c;
var _c1;
$RefreshReg$(_c, "Content$React.forwardRef");
$RefreshReg$(_c1, "Content");
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"src/Dialog/Mask.tsx": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
var _jsxdevruntime = __mako_require__("node_modules/_react@18.3.1@react/jsx-dev-runtime.js");
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react@18.3.1@react/index.js"));
var _classnames = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_classnames@2.5.1@classnames/index.js"));
var _motion = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_@rc-component_motion@1.1.3@@rc-component/motion/es/index.js"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const Mask = (props)=>{
    const { prefixCls, style, visible, maskProps, motionName, className } = props;
    return /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_motion.default, {
        visible: visible,
        motionName: motionName,
        leavedClassName: `${prefixCls}-mask-hidden`,
        children: ({ className: motionClassName, style: motionStyle }, ref)=>/*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
                ref: ref,
                style: {
                    ...motionStyle,
                    ...style
                },
                className: (0, _classnames.default)(`${prefixCls}-mask`, motionClassName, className),
                ...maskProps
            }, void 0, false, {
                fileName: "src/Dialog/Mask.tsx",
                lineNumber: 24,
                columnNumber: 9
            }, this)
    }, "mask", false, {
        fileName: "src/Dialog/Mask.tsx",
        lineNumber: 17,
        columnNumber: 5
    }, this);
};
_c = Mask;
var _default = Mask;
var _c;
$RefreshReg$(_c, "Mask");
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"src/Dialog/index.tsx": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
var _jsxdevruntime = __mako_require__("node_modules/_react@18.3.1@react/jsx-dev-runtime.js");
var _classnames = _interop_require_default._(__mako_require__("node_modules/_classnames@2.5.1@classnames/index.js"));
var _contains = _interop_require_default._(__mako_require__("node_modules/_@rc-component_util@1.2.1@@rc-component/util/lib/Dom/contains.js"));
var _useId = _interop_require_default._(__mako_require__("node_modules/_@rc-component_util@1.2.1@@rc-component/util/lib/hooks/useId.js"));
var _KeyCode = _interop_require_default._(__mako_require__("node_modules/_@rc-component_util@1.2.1@@rc-component/util/lib/KeyCode.js"));
var _pickAttrs = _interop_require_default._(__mako_require__("node_modules/_@rc-component_util@1.2.1@@rc-component/util/lib/pickAttrs.js"));
var _react = _interop_require_wildcard._(__mako_require__("node_modules/_react@18.3.1@react/index.js"));
var _util = __mako_require__("src/util.ts");
var _Content = _interop_require_default._(__mako_require__("src/Dialog/Content/index.tsx"));
var _Mask = _interop_require_default._(__mako_require__("src/Dialog/Mask.tsx"));
var _warning = __mako_require__("node_modules/_@rc-component_util@1.2.1@@rc-component/util/lib/warning.js");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _s = $RefreshSig$();
const Dialog = (props)=>{
    _s();
    const { prefixCls = 'rc-dialog', zIndex, visible = false, keyboard = true, focusTriggerAfterClose = true, wrapStyle, wrapClassName, wrapProps, onClose, afterOpenChange, afterClose, transitionName, animation, closable = true, mask = true, maskTransitionName, maskAnimation, maskClosable = true, maskStyle, maskProps, rootClassName, rootStyle, classNames: modalClassNames, styles: modalStyles } = props;
    [
        'wrapStyle',
        'bodyStyle',
        'maskStyle'
    ].forEach((prop)=>{
        (0, _warning.warning)(!(prop in props), `${prop} is deprecated, please use styles instead.`);
    });
    if ('wrapClassName' in props) (0, _warning.warning)(false, `wrapClassName is deprecated, please use classNames instead.`);
    const lastOutSideActiveElementRef = (0, _react.useRef)();
    const wrapperRef = (0, _react.useRef)();
    const contentRef = (0, _react.useRef)();
    const [animatedVisible, setAnimatedVisible] = _react.useState(visible);
    const ariaId = (0, _useId.default)();
    function saveLastOutSideActiveElementRef() {
        if (!(0, _contains.default)(wrapperRef.current, document.activeElement)) lastOutSideActiveElementRef.current = document.activeElement;
    }
    function focusDialogContent() {
        if (!(0, _contains.default)(wrapperRef.current, document.activeElement)) {
            var _contentRef_current;
            (_contentRef_current = contentRef.current) === null || _contentRef_current === void 0 || _contentRef_current.focus();
        }
    }
    function doClose() {
        setAnimatedVisible(false);
        if (mask && lastOutSideActiveElementRef.current && focusTriggerAfterClose) {
            try {
                lastOutSideActiveElementRef.current.focus({
                    preventScroll: true
                });
            } catch (e) {}
            lastOutSideActiveElementRef.current = null;
        }
        if (animatedVisible) afterClose === null || afterClose === void 0 || afterClose();
    }
    function onDialogVisibleChanged(newVisible) {
        if (newVisible) focusDialogContent();
        else doClose();
        afterOpenChange === null || afterOpenChange === void 0 || afterOpenChange(newVisible);
    }
    function onInternalClose(e) {
        onClose === null || onClose === void 0 || onClose(e);
    }
    const contentClickRef = (0, _react.useRef)(false);
    const contentTimeoutRef = (0, _react.useRef)();
    const onContentMouseDown = ()=>{
        clearTimeout(contentTimeoutRef.current);
        contentClickRef.current = true;
    };
    const onContentMouseUp = ()=>{
        contentTimeoutRef.current = setTimeout(()=>{
            contentClickRef.current = false;
        });
    };
    let onWrapperClick = null;
    if (maskClosable) onWrapperClick = (e)=>{
        if (contentClickRef.current) contentClickRef.current = false;
        else if (wrapperRef.current === e.target) onInternalClose(e);
    };
    function onWrapperKeyDown(e) {
        if (keyboard && e.keyCode === _KeyCode.default.ESC) {
            e.stopPropagation();
            onInternalClose(e);
            return;
        }
        if (visible && e.keyCode === _KeyCode.default.TAB) contentRef.current.changeActive(!e.shiftKey);
    }
    (0, _react.useEffect)(()=>{
        if (visible) {
            setAnimatedVisible(true);
            saveLastOutSideActiveElementRef();
        } else if (animatedVisible && contentRef.current.enableMotion() && !contentRef.current.inMotion()) doClose();
    }, [
        visible
    ]);
    (0, _react.useEffect)(()=>()=>{
            clearTimeout(contentTimeoutRef.current);
        }, []);
    const mergedStyle = {
        zIndex,
        ...wrapStyle,
        ...modalStyles === null || modalStyles === void 0 ? void 0 : modalStyles.wrapper,
        display: !animatedVisible ? 'none' : null
    };
    return (0, _jsxdevruntime.jsxDEV)("div", {
        className: (0, _classnames.default)(`${prefixCls}-root`, rootClassName),
        style: rootStyle,
        ...(0, _pickAttrs.default)(props, {
            data: true
        }),
        children: [
            (0, _jsxdevruntime.jsxDEV)(_Mask.default, {
                prefixCls: prefixCls,
                visible: mask && visible,
                motionName: (0, _util.getMotionName)(prefixCls, maskTransitionName, maskAnimation),
                style: {
                    zIndex,
                    ...maskStyle,
                    ...modalStyles === null || modalStyles === void 0 ? void 0 : modalStyles.mask
                },
                maskProps: maskProps,
                className: modalClassNames === null || modalClassNames === void 0 ? void 0 : modalClassNames.mask
            }, void 0, false, {
                fileName: "src/Dialog/index.tsx",
                lineNumber: 194,
                columnNumber: 7
            }, this),
            (0, _jsxdevruntime.jsxDEV)("div", {
                tabIndex: -1,
                onKeyDown: onWrapperKeyDown,
                className: (0, _classnames.default)(`${prefixCls}-wrap`, wrapClassName, modalClassNames === null || modalClassNames === void 0 ? void 0 : modalClassNames.wrapper),
                ref: wrapperRef,
                onClick: onWrapperClick,
                style: mergedStyle,
                ...wrapProps,
                children: (0, _jsxdevruntime.jsxDEV)(_Content.default, {
                    ...props,
                    onMouseDown: onContentMouseDown,
                    onMouseUp: onContentMouseUp,
                    ref: contentRef,
                    closable: closable,
                    ariaId: ariaId,
                    prefixCls: prefixCls,
                    visible: visible && animatedVisible,
                    onClose: onInternalClose,
                    onVisibleChanged: onDialogVisibleChanged,
                    motionName: (0, _util.getMotionName)(prefixCls, transitionName, animation)
                }, void 0, false, {
                    fileName: "src/Dialog/index.tsx",
                    lineNumber: 211,
                    columnNumber: 9
                }, this)
            }, void 0, false, {
                fileName: "src/Dialog/index.tsx",
                lineNumber: 202,
                columnNumber: 7
            }, this)
        ]
    }, void 0, true, {
        fileName: "src/Dialog/index.tsx",
        lineNumber: 189,
        columnNumber: 5
    }, this);
};
_s(Dialog, "jpgAGSop6biDTxPFPAMT85ShMu0=", false, function() {
    return [
        _useId.default
    ];
});
_c = Dialog;
var _default = Dialog;
var _c;
$RefreshReg$(_c, "Dialog");
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"src/DialogWrap.tsx": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
var _jsxdevruntime = __mako_require__("node_modules/_react@18.3.1@react/jsx-dev-runtime.js");
var _portal = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_@rc-component_portal@1.1.2@@rc-component/portal/es/index.js"));
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react@18.3.1@react/index.js"));
var _context = __mako_require__("src/context.ts");
var _Dialog = /*#__PURE__*/ _interop_require_default._(__mako_require__("src/Dialog/index.tsx"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _s = $RefreshSig$();
// fix issue #10656
/*
 * getContainer remarks
 * Custom container should not be return, because in the Portal component, it will remove the
 * return container element here, if the custom container is the only child of it's component,
 * like issue #10656, It will has a conflict with removeChild method in react-dom.
 * So here should add a child (div element) to custom container.
 * */ const DialogWrap = (props)=>{
    _s();
    const { visible, getContainer, forceRender, destroyOnClose = false, afterClose, panelRef } = props;
    const [animatedVisible, setAnimatedVisible] = _react.useState(visible);
    const refContext = _react.useMemo(()=>({
            panel: panelRef
        }), [
        panelRef
    ]);
    _react.useEffect(()=>{
        if (visible) setAnimatedVisible(true);
    }, [
        visible
    ]);
    // Destroy on close will remove wrapped div
    if (!forceRender && destroyOnClose && !animatedVisible) return null;
    return /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_context.RefContext.Provider, {
        value: refContext,
        children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_portal.default, {
            open: visible || forceRender || animatedVisible,
            autoDestroy: false,
            getContainer: getContainer,
            autoLock: visible || animatedVisible,
            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_Dialog.default, {
                ...props,
                destroyOnClose: destroyOnClose,
                afterClose: ()=>{
                    afterClose === null || afterClose === void 0 || afterClose();
                    setAnimatedVisible(false);
                }
            }, void 0, false, {
                fileName: "src/DialogWrap.tsx",
                lineNumber: 48,
                columnNumber: 9
            }, this)
        }, void 0, false, {
            fileName: "src/DialogWrap.tsx",
            lineNumber: 42,
            columnNumber: 7
        }, this)
    }, void 0, false, {
        fileName: "src/DialogWrap.tsx",
        lineNumber: 41,
        columnNumber: 5
    }, this);
};
_s(DialogWrap, "N2Fqxpoym3ZwLNsWl2JCF4u+lvs=");
_c = DialogWrap;
DialogWrap.displayName = 'Dialog';
var _default = DialogWrap;
var _c;
$RefreshReg$(_c, "DialogWrap");
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"src/context.ts": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "RefContext", {
    enumerable: true,
    get: function() {
        return RefContext;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react@18.3.1@react/index.js"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const RefContext = /*#__PURE__*/ _react.createContext({});
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"src/index.ts": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
function _export(target, all) {
    for(var name in all)Object.defineProperty(target, name, {
        enumerable: true,
        get: all[name]
    });
}
__mako_require__.e(exports, {
    Panel: function() {
        return _Panel.default;
    },
    default: function() {
        return _default;
    }
});
var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
var _DialogWrap = /*#__PURE__*/ _interop_require_default._(__mako_require__("src/DialogWrap.tsx"));
var _Panel = /*#__PURE__*/ _interop_require_default._(__mako_require__("src/Dialog/Content/Panel.tsx"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _default = _DialogWrap.default;
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"src/util.ts": function (module, exports, __mako_require__){
// =============================== Motion ===============================
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
function _export(target, all) {
    for(var name in all)Object.defineProperty(target, name, {
        enumerable: true,
        get: all[name]
    });
}
__mako_require__.e(exports, {
    getMotionName: function() {
        return getMotionName;
    },
    offset: function() {
        return offset;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
function getMotionName(prefixCls, transitionName, animationName) {
    let motionName = transitionName;
    if (!motionName && animationName) motionName = `${prefixCls}-${animationName}`;
    return motionName;
}
// =============================== Offset ===============================
function getScroll(w, top) {
    let ret = w[`page${top ? 'Y' : 'X'}Offset`];
    const method = `scroll${top ? 'Top' : 'Left'}`;
    if (typeof ret !== 'number') {
        const d = w.document;
        ret = d.documentElement[method];
        if (typeof ret !== 'number') ret = d.body[method];
    }
    return ret;
}
function offset(el) {
    const rect = el.getBoundingClientRect();
    const pos = {
        left: rect.left,
        top: rect.top
    };
    const doc = el.ownerDocument;
    const w = doc.defaultView || doc.parentWindow;
    pos.left += getScroll(w);
    pos.top += getScroll(w, true);
    return pos;
}
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
 }]);
