// @rc-component/dialog/dist/demos-async.7338948553039754745.hot-update.js
globalThis.makoModuleHotUpdate('demos', {
    modules: {
        "docs/examples/multiple-Portal.tsx?watch=parent": function(module, exports, __mako_require__) {
            "use strict";
            __mako_require__.d(exports, "__esModule", {
                value: true
            });
            var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
            var _reactrefresh = _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
            var prevRefreshReg;
            var prevRefreshSig;
            prevRefreshReg = self.$RefreshReg$;
            prevRefreshSig = self.$RefreshSig$;
            self.$RefreshReg$ = (type, id)=>{
                _reactrefresh.register(type, module.id + id);
            };
            self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
            if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
            if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
            function registerClassComponent(filename, moduleExports) {
                for(const key in moduleExports)try {
                    if (key === "__esModule") continue;
                    const exportValue = moduleExports[key];
                    if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
                } catch (e) {}
            }
            function $RefreshIsReactComponentLike$(moduleExports) {
                if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
                for(var key in moduleExports)try {
                    if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
                } catch (e) {}
                return false;
            }
            registerClassComponent(module.id, module.exports);
            if ($RefreshIsReactComponentLike$(module.exports)) {
                module.meta.hot.accept();
                _reactrefresh.performReactRefresh();
            }
        },
        "docs/examples/multiple-Portal.tsx?dumi-raw": function(module, exports, __mako_require__) {
            "use strict";
            __mako_require__.d(exports, "__esModule", {
                value: true
            });
            __mako_require__.d(exports, "default", {
                enumerable: true,
                get: function() {
                    return _default;
                }
            });
            var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
            var _reactrefresh = _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
            __mako_require__("docs/examples/multiple-Portal.tsx?watch=parent");
            var prevRefreshReg;
            var prevRefreshSig;
            prevRefreshReg = self.$RefreshReg$;
            prevRefreshSig = self.$RefreshSig$;
            self.$RefreshReg$ = (type, id)=>{
                _reactrefresh.register(type, module.id + id);
            };
            self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
            var _default = "import * as React from 'react';\nimport Drawer from 'rc-drawer';\nimport 'rc-drawer/assets/index.css';\nimport Dialog from 'rc-dialog';\nimport '../../assets/index.less';\n\nconst Demo: React.FC = () => {\n  const [showDialog, setShowDialog] = React.useState(false);\n  const [showDrawer, setShowDrawer] = React.useState(false);\n\n  const onToggleDrawer = () => {\n    setShowDrawer((value) => !value);\n  };\n\n  const onToggleDialog = () => {\n    setShowDialog((value) => !value);\n  };\n\n  const dialog = (\n    <Dialog\n      visible={showDialog}\n      animation=\"zoom\"\n      maskAnimation=\"fade\"\n      onClose={onToggleDialog}\n      forceRender\n      title=\"basic modal\"\n    >\n      <p>\n        <button type=\"button\" onClick={onToggleDrawer}>\n          show drawer\n        </button>\n      </p>\n      <div style={{ height: 200 }} />\n    </Dialog>\n  );\n  const drawer = (\n    <Drawer open={showDrawer} onClose={onToggleDrawer}>\n      <button type=\"button\" onClick={onToggleDrawer}>\n        close drawer\n      </button>\n    </Drawer>\n  );\n  return (\n    <div>\n      <button type=\"button\" onClick={onToggleDialog}>\n        open dialog\n      </button>\n      <button\n        type=\"button\"\n        onClick={() => {\n          setShowDialog(true);\n          setTimeout(() => {\n            setShowDialog(false);\n          }, 0);\n        }}\n      >\n        quick\n      </button>\n      {dialog}\n      {drawer}\n    </div>\n  );\n};\n\nexport default Demo;\n";
            if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
            if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
            function registerClassComponent(filename, moduleExports) {
                for(const key in moduleExports)try {
                    if (key === "__esModule") continue;
                    const exportValue = moduleExports[key];
                    if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
                } catch (e) {}
            }
            function $RefreshIsReactComponentLike$(moduleExports) {
                if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
                for(var key in moduleExports)try {
                    if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
                } catch (e) {}
                return false;
            }
            registerClassComponent(module.id, module.exports);
            if ($RefreshIsReactComponentLike$(module.exports)) {
                module.meta.hot.accept();
                _reactrefresh.performReactRefresh();
            }
        },
        "docs/examples/multiple-Portal.tsx?techStack=react": function(module, exports, __mako_require__) {
            "use strict";
            __mako_require__.d(exports, "__esModule", {
                value: true
            });
            __mako_require__.d(exports, "default", {
                enumerable: true,
                get: function() {
                    return _default;
                }
            });
            var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
            var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
            var _reactrefresh = _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
            var _jsxdevruntime = __mako_require__("node_modules/_react@18.3.1@react/jsx-dev-runtime.js");
            __mako_require__("docs/examples/multiple-Portal.tsx?watch=parent");
            var _react = _interop_require_wildcard._(__mako_require__("node_modules/_react@18.3.1@react/index.js"));
            var _rcdrawer = _interop_require_default._(__mako_require__("node_modules/_rc-drawer@7.2.0@rc-drawer/es/index.js"));
            "";
            var _rcdialog = _interop_require_default._(__mako_require__("src/index.ts"));
            "";
            var prevRefreshReg;
            var prevRefreshSig;
            prevRefreshReg = self.$RefreshReg$;
            prevRefreshSig = self.$RefreshSig$;
            self.$RefreshReg$ = (type, id)=>{
                _reactrefresh.register(type, module.id + id);
            };
            self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
            var _s = $RefreshSig$();
            const Demo = ()=>{
                _s();
                const [showDialog, setShowDialog] = _react.useState(false);
                const [showDrawer, setShowDrawer] = _react.useState(false);
                const onToggleDrawer = ()=>{
                    setShowDrawer((value)=>!value);
                };
                const onToggleDialog = ()=>{
                    setShowDialog((value)=>!value);
                };
                const dialog = (0, _jsxdevruntime.jsxDEV)(_rcdialog.default, {
                    visible: showDialog,
                    animation: "zoom",
                    maskAnimation: "fade",
                    onClose: onToggleDialog,
                    forceRender: true,
                    title: "basic modal",
                    children: [
                        (0, _jsxdevruntime.jsxDEV)("p", {
                            children: (0, _jsxdevruntime.jsxDEV)("button", {
                                type: "button",
                                onClick: onToggleDrawer,
                                children: "show drawer"
                            }, void 0, false, {
                                fileName: "docs/examples/multiple-Portal.tsx?techStack=react",
                                lineNumber: 29,
                                columnNumber: 9
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/examples/multiple-Portal.tsx?techStack=react",
                            lineNumber: 28,
                            columnNumber: 7
                        }, this),
                        (0, _jsxdevruntime.jsxDEV)("div", {
                            style: {
                                height: 200
                            }
                        }, void 0, false, {
                            fileName: "docs/examples/multiple-Portal.tsx?techStack=react",
                            lineNumber: 33,
                            columnNumber: 7
                        }, this)
                    ]
                }, void 0, true, {
                    fileName: "docs/examples/multiple-Portal.tsx?techStack=react",
                    lineNumber: 20,
                    columnNumber: 5
                }, this);
                const drawer = (0, _jsxdevruntime.jsxDEV)(_rcdrawer.default, {
                    open: showDrawer,
                    onClose: onToggleDrawer,
                    children: (0, _jsxdevruntime.jsxDEV)("button", {
                        type: "button",
                        onClick: onToggleDrawer,
                        children: "close drawer"
                    }, void 0, false, {
                        fileName: "docs/examples/multiple-Portal.tsx?techStack=react",
                        lineNumber: 38,
                        columnNumber: 7
                    }, this)
                }, void 0, false, {
                    fileName: "docs/examples/multiple-Portal.tsx?techStack=react",
                    lineNumber: 37,
                    columnNumber: 5
                }, this);
                return (0, _jsxdevruntime.jsxDEV)("div", {
                    children: [
                        (0, _jsxdevruntime.jsxDEV)("button", {
                            type: "button",
                            onClick: onToggleDialog,
                            children: "open dialog"
                        }, void 0, false, {
                            fileName: "docs/examples/multiple-Portal.tsx?techStack=react",
                            lineNumber: 45,
                            columnNumber: 7
                        }, this),
                        (0, _jsxdevruntime.jsxDEV)("button", {
                            type: "button",
                            onClick: ()=>{
                                setShowDialog(true);
                                setTimeout(()=>{
                                    setShowDialog(false);
                                }, 0);
                            },
                            children: "quick"
                        }, void 0, false, {
                            fileName: "docs/examples/multiple-Portal.tsx?techStack=react",
                            lineNumber: 48,
                            columnNumber: 7
                        }, this),
                        dialog,
                        drawer
                    ]
                }, void 0, true, {
                    fileName: "docs/examples/multiple-Portal.tsx?techStack=react",
                    lineNumber: 44,
                    columnNumber: 5
                }, this);
            };
            _s(Demo, "NXmLaLBTsMoIATQhEKlsH30qgiI=");
            _c = Demo;
            var _default = Demo;
            var _c;
            $RefreshReg$(_c, "Demo");
            if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
            if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
            function registerClassComponent(filename, moduleExports) {
                for(const key in moduleExports)try {
                    if (key === "__esModule") continue;
                    const exportValue = moduleExports[key];
                    if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
                } catch (e) {}
            }
            function $RefreshIsReactComponentLike$(moduleExports) {
                if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
                for(var key in moduleExports)try {
                    if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
                } catch (e) {}
                return false;
            }
            registerClassComponent(module.id, module.exports);
            if ($RefreshIsReactComponentLike$(module.exports)) {
                module.meta.hot.accept();
                _reactrefresh.performReactRefresh();
            }
        }
    }
}, function(runtime) {
    runtime._h = '8742625841545363477';
    runtime.updateEnsure2Map({
        ".dumi/tmp/dumi/theme/ContextWrapper.tsx": [
            "dumi__tmp__dumi__theme__ContextWrapper"
        ],
        "docs/changelog.md": [
            "vendors",
            "docs/changelog.md"
        ],
        "docs/changelog.md?type=demo": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/changelog.md?type=text": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/demo/ant-design.md": [
            "vendors",
            "docs/demo/ant-design.md"
        ],
        "docs/demo/ant-design.md?type=demo": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/demo/ant-design.md?type=text": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/demo/bootstrap.md": [
            "vendors",
            "docs/demo/bootstrap.md"
        ],
        "docs/demo/bootstrap.md?type=demo": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/demo/bootstrap.md?type=text": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/demo/draggable.md": [
            "vendors",
            "docs/demo/draggable.md"
        ],
        "docs/demo/draggable.md?type=demo": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/demo/draggable.md?type=text": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/demo/multiple-Portal.md": [
            "vendors",
            "docs/demo/multiple-Portal.md"
        ],
        "docs/demo/multiple-Portal.md?type=demo": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/demo/multiple-Portal.md?type=text": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/demo/pure.md": [
            "vendors",
            "docs/demo/pure.md"
        ],
        "docs/demo/pure.md?type=demo": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/demo/pure.md?type=text": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/examples/ant-design.tsx?techStack=react": [
            "vendors",
            "common",
            "demos"
        ],
        "docs/examples/bootstrap.tsx?techStack=react": [
            "vendors",
            "common",
            "demos"
        ],
        "docs/examples/draggable.tsx?techStack=react": [
            "vendors",
            "common",
            "demos"
        ],
        "docs/examples/multiple-Portal.tsx?techStack=react": [
            "vendors",
            "common",
            "demos"
        ],
        "docs/examples/pure.tsx?techStack=react": [
            "vendors",
            "common",
            "demos"
        ],
        "docs/index.md": [
            "vendors",
            "docs/index.md"
        ],
        "docs/index.md?type=demo": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "docs/index.md?type=text": [
            "vendors",
            "common",
            "meta__docs"
        ],
        "node_modules/_dumi@2.4.17@dumi/dist/client/misc/reactDemoCompiler.js": [
            "vendors",
            "node_modules/_dumi@2.4.17@dumi/dist/client/misc/reactDemoCompiler.js"
        ],
        "node_modules/_dumi@2.4.17@dumi/dist/client/pages/404.js": [
            "vendors",
            "nm__dumi__dist__client__pages__404"
        ],
        "node_modules/_dumi@2.4.17@dumi/dist/client/pages/Demo/index.js": [
            "vendors",
            "nm__dumi__dist__client__pages__Demo__index"
        ],
        "node_modules/_dumi@2.4.17@dumi/theme-default/layouts/DocLayout/index.js": [
            "vendors",
            "node_modules/_dumi@2.4.17@dumi/theme-default/layouts/DocLayout/index.js"
        ],
        "node_modules/_react-dom@18.3.1@react-dom/server.browser.js": [
            "vendors",
            "node_modules/_react-dom@18.3.1@react-dom/server.browser.js"
        ]
    });
    ;
});
