// @rc-component/dialog/dist/docs_changelog_md-async.js
((typeof globalThis !== 'undefined' ? globalThis : self)["makoChunk_@rc-component/dialog"] = (typeof globalThis !== 'undefined' ? globalThis : self)["makoChunk_@rc-component/dialog"] || []).push([
        ['docs/changelog.md'],
{ "CHANGELOG.md?watch=parent": function (module, exports, __mako_require__){

},
"docs/changelog.md": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
var _jsxdevruntime = __mako_require__("node_modules/_react@18.3.1@react/jsx-dev-runtime.js");
var _index = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_dumi@2.4.17@dumi/theme-default/builtins/API/index.js"));
var _index1 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_dumi@2.4.17@dumi/theme-default/builtins/Badge/index.js"));
var _index2 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_dumi@2.4.17@dumi/theme-default/builtins/CodeGroup/index.js"));
var _index3 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_dumi@2.4.17@dumi/theme-default/builtins/Container/index.js"));
var _index4 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_dumi@2.4.17@dumi/theme-default/builtins/Previewer/index.js"));
var _index5 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_dumi@2.4.17@dumi/theme-default/builtins/SourceCode/index.js"));
var _index6 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_dumi@2.4.17@dumi/theme-default/builtins/Table/index.js"));
var _index7 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_dumi@2.4.17@dumi/theme-default/builtins/Tree/index.js"));
var _dumi = __mako_require__(".dumi/tmp/dumi/exports.ts");
__mako_require__("CHANGELOG.md?watch=parent");
var _loading = /*#__PURE__*/ _interop_require_default._(__mako_require__(".dumi/tmp/dumi/theme/loading.tsx"));
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react@18.3.1@react/index.js"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _s = $RefreshSig$();
function DumiMarkdownInner() {
    _s();
    const { texts: $$contentTexts } = (0, _dumi.useRouteMeta)();
    return /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_jsxdevruntime.Fragment, {
        children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
            className: "markdown",
            children: [
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h1", {
                    id: "changelog",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#changelog",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 111
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 57
                        }, this),
                        "Changelog"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 38
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "851",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#851",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 229
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 181
                        }, this),
                        "8.5.1"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 168
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                        children: $$contentTexts[0].value
                    }, void 0, false, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 285
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 282
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: [
                            $$contentTexts[1].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                href: "https://github.com/react-component/dialog/pull/227",
                                children: $$contentTexts[2].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 360
                            }, this)
                        ]
                    }, void 0, true, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 331
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 327
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "850",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#850",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 521
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 473
                        }, this),
                        "8.5.0"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 460
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                        children: $$contentTexts[3].value
                    }, void 0, false, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 577
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 574
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: [
                            $$contentTexts[4].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                href: "https://github.com/react-component/dialog/pull/220",
                                children: $$contentTexts[5].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 652
                            }, this)
                        ]
                    }, void 0, true, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 623
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 619
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "846",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#846",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 813
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 765
                        }, this),
                        "8.4.6"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 752
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                        children: $$contentTexts[6].value
                    }, void 0, false, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 869
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 866
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: [
                            $$contentTexts[7].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                href: "https://github.com/react-component/dialog/pull/218",
                                children: $$contentTexts[8].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 944
                            }, this)
                        ]
                    }, void 0, true, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 915
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 911
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "845",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#845",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 1105
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 1057
                        }, this),
                        "8.4.5"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 1044
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                        children: $$contentTexts[9].value
                    }, void 0, false, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 1161
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 1158
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: [
                            $$contentTexts[10].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                href: "https://github.com/react-component/dialog/pull/217",
                                children: $$contentTexts[11].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 1237
                            }, this)
                        ]
                    }, void 0, true, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 1207
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 1203
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "844",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#844",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 1399
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 1351
                        }, this),
                        "8.4.4"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 1338
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                        children: $$contentTexts[12].value
                    }, void 0, false, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 1455
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 1452
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                                children: [
                                    $$contentTexts[13].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                        href: "https://github.com/react-component/dialog/pull/210",
                                        children: $$contentTexts[14].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 22,
                                        columnNumber: 1535
                                    }, this)
                                ]
                            }, void 0, true, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 1506
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 1502
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                                children: [
                                    $$contentTexts[15].value,
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                        href: "https://github.com/react-component/dialog/pull/212",
                                        children: $$contentTexts[16].value
                                    }, void 0, false, {
                                        fileName: "docs/changelog.md",
                                        lineNumber: 22,
                                        columnNumber: 1668
                                    }, this)
                                ]
                            }, void 0, true, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 1639
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 1635
                        }, this)
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 1498
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "843",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#843",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 1834
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 1786
                        }, this),
                        "8.4.3"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 1773
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                        children: $$contentTexts[17].value
                    }, void 0, false, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 1890
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 1887
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: [
                            $$contentTexts[18].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                href: "https://github.com/react-component/dialog/pull/207",
                                children: $$contentTexts[19].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 1967
                            }, this)
                        ]
                    }, void 0, true, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 1937
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 1933
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "842",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#842",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 2129
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 2081
                        }, this),
                        "8.4.2"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 2068
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                        children: $$contentTexts[20].value
                    }, void 0, false, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 2185
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 2182
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: [
                            $$contentTexts[21].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                href: "https://github.com/react-component/dialog/pull/206",
                                children: $$contentTexts[22].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 2262
                            }, this)
                        ]
                    }, void 0, true, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 2232
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 2228
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "841",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#841",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 2424
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 2376
                        }, this),
                        "8.4.1"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 2363
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                        children: $$contentTexts[23].value
                    }, void 0, false, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 2480
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 2477
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: [
                            $$contentTexts[24].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                href: "https://github.com/react-component/dialog/pull/204",
                                children: $$contentTexts[25].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 2557
                            }, this)
                        ]
                    }, void 0, true, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 2527
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 2523
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "840",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#840",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 2719
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 2671
                        }, this),
                        "8.4.0"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 2658
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                        children: $$contentTexts[26].value
                    }, void 0, false, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 2775
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 2772
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: [
                            $$contentTexts[27].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                children: $$contentTexts[28].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 2852
                            }, this),
                            $$contentTexts[29].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                href: "https://github.com/react-component/dialog/pull/203",
                                children: $$contentTexts[30].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 2917
                            }, this)
                        ]
                    }, void 0, true, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 2822
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 2818
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "834-822-812--2020-09-04",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#834-822-812--2020-09-04",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 3119
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 3051
                        }, this),
                        "8.3.4 (8.2.2, 8.1.2) / 2020-09-04"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 3018
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: [
                            $$contentTexts[31].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                href: "https://github.com/ant-design/ant-design/issues/26582",
                                children: $$contentTexts[32].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 3234
                            }, this)
                        ]
                    }, void 0, true, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 3204
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 3200
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "833--2020-09-02",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#833--2020-09-02",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 3423
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 3363
                        }, this),
                        "8.3.3 / 2020-09-02"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 3338
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: [
                            $$contentTexts[33].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                href: "https://github.com/react-component/dialog/pull/202",
                                children: $$contentTexts[34].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 3523
                            }, this)
                        ]
                    }, void 0, true, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 3493
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 3489
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "832--2020-09-02",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#832--2020-09-02",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 3709
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 3649
                        }, this),
                        "8.3.2 / 2020-09-02"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 3624
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: [
                            $$contentTexts[35].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                href: "https://github.com/react-component/dialog/pull/201",
                                children: $$contentTexts[36].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 3809
                            }, this)
                        ]
                    }, void 0, true, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 3779
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 3775
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "831--2020-09-02",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#831--2020-09-02",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 3995
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 3935
                        }, this),
                        "8.3.1 / 2020-09-02"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 3910
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: [
                            $$contentTexts[37].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                href: "https://github.com/react-component/dialog/pull/200",
                                children: $$contentTexts[38].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 4095
                            }, this)
                        ]
                    }, void 0, true, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 4065
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 4061
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "830--2020-08-31",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#830--2020-08-31",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 4281
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 4221
                        }, this),
                        "8.3.0 / 2020-08-31"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 4196
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: [
                            $$contentTexts[39].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                children: $$contentTexts[40].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 4381
                            }, this),
                            $$contentTexts[41].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                href: "https://github.com/react-component/dialog/pull/195",
                                children: $$contentTexts[42].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 4446
                            }, this)
                        ]
                    }, void 0, true, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 4351
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 4347
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "820--2020-08-27",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#820--2020-08-27",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 4632
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 4572
                        }, this),
                        "8.2.0 / 2020-08-27"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 4547
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: [
                            $$contentTexts[43].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                children: $$contentTexts[44].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 4732
                            }, this),
                            $$contentTexts[45].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                href: "https://github.com/react-component/dialog/pull/197",
                                children: $$contentTexts[46].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 4797
                            }, this)
                        ]
                    }, void 0, true, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 4702
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 4698
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "811--2020-08-19",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#811--2020-08-19",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 4983
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 4923
                        }, this),
                        "8.1.1 / 2020-08-19"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 4898
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: [
                            $$contentTexts[47].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                href: "https://github.com/react-component/dialog/pull/194",
                                children: $$contentTexts[48].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 5083
                            }, this)
                        ]
                    }, void 0, true, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 5053
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 5049
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "810--2020-07-09",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#810--2020-07-09",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 5269
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 5209
                        }, this),
                        "8.1.0 / 2020-07-09"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 5184
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                            children: $$contentTexts[49].value
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 5339
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                            children: [
                                $$contentTexts[50].value,
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                    children: $$contentTexts[51].value
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 22,
                                    columnNumber: 5404
                                }, this),
                                $$contentTexts[52].value,
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                    children: $$contentTexts[53].value
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 22,
                                    columnNumber: 5469
                                }, this),
                                $$contentTexts[54].value
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 5374
                        }, this)
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 5335
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "800--2020-05-29",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#800--2020-05-29",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 5629
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 5569
                        }, this),
                        "8.0.0 / 2020-05-29"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 5544
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: [
                            $$contentTexts[55].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                children: $$contentTexts[56].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 5729
                            }, this),
                            $$contentTexts[57].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                children: $$contentTexts[58].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 5794
                            }, this),
                            $$contentTexts[59].value
                        ]
                    }, void 0, true, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 5699
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 5695
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "770--2020-05-05",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#770--2020-05-05",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 5954
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 5894
                        }, this),
                        "7.7.0 / 2020-05-05"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 5869
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: [
                            $$contentTexts[60].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                children: $$contentTexts[61].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 6054
                            }, this),
                            $$contentTexts[62].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                children: $$contentTexts[63].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 6119
                            }, this),
                            $$contentTexts[64].value
                        ]
                    }, void 0, true, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 6024
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 6020
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "740--2019-05-10",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#740--2019-05-10",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 6279
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 6219
                        }, this),
                        "7.4.0 / 2019-05-10"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 6194
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: $$contentTexts[65].value
                    }, void 0, false, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 6349
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 6345
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "730--2018-12-06",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#730--2018-12-06",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 6474
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 6414
                        }, this),
                        "7.3.0 / 2018-12-06"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 6389
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: [
                            $$contentTexts[66].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                                children: $$contentTexts[67].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 6574
                            }, this),
                            $$contentTexts[68].value
                        ]
                    }, void 0, true, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 6544
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 6540
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "720--2018-07-30",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#720--2018-07-30",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 6734
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 6674
                        }, this),
                        "7.2.0 / 2018-07-30"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 6649
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: [
                            $$contentTexts[69].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_dumi.Link, {
                                to: "/*https://github.com/react-component/dialog/pull/89",
                                children: $$contentTexts[70].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 6834
                            }, this),
                            $$contentTexts[71].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                href: "https://github.com/HeskeyBaozi",
                                children: $$contentTexts[72].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 6956
                            }, this)
                        ]
                    }, void 0, true, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 6804
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 6800
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "710--2017-12-28",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#710--2017-12-28",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 7122
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 7062
                        }, this),
                        "7.1.0 / 2017-12-28"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 7037
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: [
                            $$contentTexts[73].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                href: "https://github.com/react-component/dialog/pull/72",
                                children: $$contentTexts[74].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 7222
                            }, this),
                            $$contentTexts[75].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                href: "https://github.com/Rohanhacker",
                                children: $$contentTexts[76].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 7338
                            }, this)
                        ]
                    }, void 0, true, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 7192
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 7188
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "700--2017-11-02",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#700--2017-11-02",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 7504
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 7444
                        }, this),
                        "7.0.0 / 2017-11-02"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 7419
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                            children: [
                                $$contentTexts[77].value,
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                    href: "https://github.com/react-component/m-dialog",
                                    children: $$contentTexts[78].value
                                }, void 0, false, {
                                    fileName: "docs/changelog.md",
                                    lineNumber: 22,
                                    columnNumber: 7604
                                }, this),
                                $$contentTexts[79].value
                            ]
                        }, void 0, true, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 7574
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                            children: $$contentTexts[80].value
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 7719
                        }, this)
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 7570
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: [
                        $$contentTexts[81].value,
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            href: "https://github.com/facebook/react/issues/10826",
                            children: $$contentTexts[82].value
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 7788
                        }, this)
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 7759
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "6511--2017-8-21",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#6511--2017-8-21",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 7964
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 7904
                        }, this),
                        "6.5.11 / 2017-8-21"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 7879
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: [
                            $$contentTexts[83].value,
                            /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                                href: "https://github.com/react-component/dialog/pull/64",
                                children: $$contentTexts[84].value
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 8064
                            }, this)
                        ]
                    }, void 0, true, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 8034
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 8030
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "650--2016-10-25",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#650--2016-10-25",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 8249
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 8189
                        }, this),
                        "6.5.0 / 2016-10-25"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 8164
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: $$contentTexts[85].value
                    }, void 0, false, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 8319
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 8315
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "640--2016-09-19",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#640--2016-09-19",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 8444
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 8384
                        }, this),
                        "6.4.0 / 2016-09-19"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 8359
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: $$contentTexts[86].value
                    }, void 0, false, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 8514
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 8510
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "620--2016-07-18",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#620--2016-07-18",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 8639
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 8579
                        }, this),
                        "6.2.0 / 2016-07-18"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 8554
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: $$contentTexts[87].value
                    }, void 0, false, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 8709
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 8705
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "600--2016-03-18",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#600--2016-03-18",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 8834
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 8774
                        }, this),
                        "6.0.0 / 2016-03-18"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 8749
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                            children: $$contentTexts[88].value
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 8904
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                            children: $$contentTexts[89].value
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 8939
                        }, this)
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 8900
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "540--2016-02-27",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#540--2016-02-27",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 9064
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 9004
                        }, this),
                        "5.4.0 / 2016-02-27"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 8979
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: $$contentTexts[90].value
                    }, void 0, false, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 9134
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 9130
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "530--2015-11-23",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#530--2015-11-23",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 9259
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 9199
                        }, this),
                        "5.3.0 / 2015-11-23"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 9174
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: $$contentTexts[91].value
                    }, void 0, false, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 9329
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 9325
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "510--2015-10-20",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#510--2015-10-20",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 9454
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 9394
                        }, this),
                        "5.1.0 / 2015-10-20"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 9369
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: $$contentTexts[92].value
                    }, void 0, false, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 9524
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 9520
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "500--2015-08-17",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#500--2015-08-17",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 9649
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 9589
                        }, this),
                        "5.0.0 / 2015-08-17"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 9564
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("ul", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("li", {
                        children: $$contentTexts[93].value
                    }, void 0, false, {
                        fileName: "docs/changelog.md",
                        lineNumber: 22,
                        columnNumber: 9719
                    }, this)
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 9715
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "450--2015-07-23",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#450--2015-07-23",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 9844
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 9784
                        }, this),
                        "4.5.0 / 2015-07-23"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 9759
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: $$contentTexts[94].value
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 9910
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "440--2015-07-03",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#440--2015-07-03",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 10028
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 9968
                        }, this),
                        "4.4.0 / 2015-07-03"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 9943
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: $$contentTexts[95].value
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 10094
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "420--2015-06-09",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#420--2015-06-09",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 10212
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 10152
                        }, this),
                        "4.2.0 / 2015-06-09"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 10127
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: $$contentTexts[96].value
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 10278
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "400--2015-04-28",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#400--2015-04-28",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 10396
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 10336
                        }, this),
                        "4.0.0 / 2015-04-28"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 10311
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: [
                        $$contentTexts[97].value,
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            href: "https://github.com/yiminghe/dom-align",
                            children: $$contentTexts[98].value
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 10491
                        }, this),
                        $$contentTexts[99].value
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 10462
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "300--2015-03-17",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#300--2015-03-17",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 10684
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 10624
                        }, this),
                        "3.0.0 / 2015-03-17"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 10599
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: $$contentTexts[100].value
                }, void 0, false, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 10750
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "210--2015-03-05",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#210--2015-03-05",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/changelog.md",
                                lineNumber: 22,
                                columnNumber: 10869
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 10809
                        }, this),
                        "2.1.0 / 2015-03-05"
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 10784
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("code", {
                            children: $$contentTexts[101].value
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 10938
                        }, this),
                        $$contentTexts[102].value,
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            href: "https://github.com/react-component/dialog/issues/3",
                            children: $$contentTexts[103].value
                        }, void 0, false, {
                            fileName: "docs/changelog.md",
                            lineNumber: 22,
                            columnNumber: 11005
                        }, this),
                        $$contentTexts[104].value
                    ]
                }, void 0, true, {
                    fileName: "docs/changelog.md",
                    lineNumber: 22,
                    columnNumber: 10935
                }, this)
            ]
        }, void 0, true, {
            fileName: "docs/changelog.md",
            lineNumber: 22,
            columnNumber: 12
        }, this)
    }, void 0, false);
}
_s(DumiMarkdownInner, "EsmAA1S45Kg2SXhnbajAzasObB8=", false, function() {
    return [
        _dumi.useRouteMeta
    ];
});
_c = DumiMarkdownInner;
// export named function for fastRefresh
// ref: https://github.com/pmmmwh/react-refresh-webpack-plugin/blob/main/docs/TROUBLESHOOTING.md#edits-always-lead-to-full-reload
function DumiMarkdownContent() {
    // wrap suspense for catch async meta data
    return /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_dumi.DumiPage, {
        children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_react.Suspense, {
            fallback: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_loading.default, {}, void 0, false, {
                fileName: "docs/changelog.md",
                lineNumber: 29,
                columnNumber: 40
            }, void 0),
            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(DumiMarkdownInner, {}, void 0, false, {
                fileName: "docs/changelog.md",
                lineNumber: 29,
                columnNumber: 62
            }, this)
        }, void 0, false, {
            fileName: "docs/changelog.md",
            lineNumber: 29,
            columnNumber: 20
        }, this)
    }, void 0, false, {
        fileName: "docs/changelog.md",
        lineNumber: 29,
        columnNumber: 10
    }, this);
}
_c1 = DumiMarkdownContent;
var _default = DumiMarkdownContent;
var _c;
var _c1;
$RefreshReg$(_c, "DumiMarkdownInner");
$RefreshReg$(_c1, "DumiMarkdownContent");
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
 }]);
