// @rc-component/dialog/dist/docs_index_md-async.js
((typeof globalThis !== 'undefined' ? globalThis : self)["makoChunk_@rc-component/dialog"] = (typeof globalThis !== 'undefined' ? globalThis : self)["makoChunk_@rc-component/dialog"] || []).push([
        ['docs/index.md'],
{ "README.md?watch=parent": function (module, exports, __mako_require__){

},
"docs/index.md": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
var _jsxdevruntime = __mako_require__("node_modules/_react@18.3.1@react/jsx-dev-runtime.js");
var _index = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_dumi@2.4.17@dumi/theme-default/builtins/API/index.js"));
var _index1 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_dumi@2.4.17@dumi/theme-default/builtins/Badge/index.js"));
var _index2 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_dumi@2.4.17@dumi/theme-default/builtins/CodeGroup/index.js"));
var _index3 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_dumi@2.4.17@dumi/theme-default/builtins/Container/index.js"));
var _index4 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_dumi@2.4.17@dumi/theme-default/builtins/Previewer/index.js"));
var _index5 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_dumi@2.4.17@dumi/theme-default/builtins/SourceCode/index.js"));
var _index6 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_dumi@2.4.17@dumi/theme-default/builtins/Table/index.js"));
var _index7 = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_dumi@2.4.17@dumi/theme-default/builtins/Tree/index.js"));
var _dumi = __mako_require__(".dumi/tmp/dumi/exports.ts");
__mako_require__("README.md?watch=parent");
var _loading = /*#__PURE__*/ _interop_require_default._(__mako_require__(".dumi/tmp/dumi/theme/loading.tsx"));
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react@18.3.1@react/index.js"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _s = $RefreshSig$();
function DumiMarkdownInner() {
    _s();
    const { texts: $$contentTexts } = (0, _dumi.useRouteMeta)();
    return /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_jsxdevruntime.Fragment, {
        children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("div", {
            className: "markdown",
            children: [
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h1", {
                    id: "rc-dialog",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#rc-dialog",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/index.md",
                                lineNumber: 22,
                                columnNumber: 111
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 22,
                            columnNumber: 57
                        }, this),
                        "rc-dialog"
                    ]
                }, void 0, true, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 38
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: $$contentTexts[0].value
                }, void 0, false, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 168
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            href: "http://npmjs.org/package/rc-dialog",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("img", {
                                src: "http://img.shields.io/npm/v/rc-dialog.svg?style=flat-square",
                                alt: "NPM version"
                            }, void 0, false, {
                                fileName: "docs/index.md",
                                lineNumber: 22,
                                columnNumber: 248
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 22,
                            columnNumber: 203
                        }, this),
                        $$contentTexts[1].value,
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            href: "https://github.com/umijs/dumi",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("img", {
                                src: "https://img.shields.io/badge/docs%20by-dumi-blue?style=flat-square",
                                alt: "dumi"
                            }, void 0, false, {
                                fileName: "docs/index.md",
                                lineNumber: 22,
                                columnNumber: 408
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 22,
                            columnNumber: 368
                        }, this),
                        $$contentTexts[2].value,
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            href: "https://github.com/react-component/dialog/actions",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("img", {
                                src: "https://github.com/react-component/dialog/workflows/CI/badge.svg",
                                alt: "build status"
                            }, void 0, false, {
                                fileName: "docs/index.md",
                                lineNumber: 22,
                                columnNumber: 588
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 22,
                            columnNumber: 528
                        }, this),
                        $$contentTexts[3].value,
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            href: "https://app.codecov.io/gh/react-component/dialog",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("img", {
                                src: "https://img.shields.io/codecov/c/github/react-component/dialog/master.svg?style=flat-square",
                                alt: "Test coverage"
                            }, void 0, false, {
                                fileName: "docs/index.md",
                                lineNumber: 22,
                                columnNumber: 773
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 22,
                            columnNumber: 714
                        }, this),
                        $$contentTexts[4].value,
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            href: "https://npmjs.org/package/rc-dialog",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("img", {
                                src: "https://img.shields.io/npm/dm/rc-dialog.svg?style=flat-square",
                                alt: "npm download"
                            }, void 0, false, {
                                fileName: "docs/index.md",
                                lineNumber: 22,
                                columnNumber: 973
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 22,
                            columnNumber: 927
                        }, this),
                        $$contentTexts[5].value,
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            href: "https://bundlephobia.com/result?p=rc-dialog",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("img", {
                                src: "https://badgen.net/bundlephobia/minzip/rc-dialog",
                                alt: "bundle size"
                            }, void 0, false, {
                                fileName: "docs/index.md",
                                lineNumber: 22,
                                columnNumber: 1150
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 22,
                            columnNumber: 1096
                        }, this)
                    ]
                }, void 0, true, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 200
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "screenshot",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#screenshot",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/index.md",
                                lineNumber: 22,
                                columnNumber: 1313
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 22,
                            columnNumber: 1258
                        }, this),
                        "Screenshot"
                    ]
                }, void 0, true, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 1238
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("img", {
                    src: "http://gtms04.alicdn.com/tps/i4/TB1dp5lHXXXXXbmXpXXyVug.FXX-664-480.png"
                }, void 0, false, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 1371
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "example",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#example",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/index.md",
                                lineNumber: 22,
                                columnNumber: 1525
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 22,
                            columnNumber: 1473
                        }, this),
                        "Example"
                    ]
                }, void 0, true, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 1456
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                        href: "http://localhost:8007/examples/",
                        children: $$contentTexts[6].value
                    }, void 0, false, {
                        fileName: "docs/index.md",
                        lineNumber: 22,
                        columnNumber: 1583
                    }, this)
                }, void 0, false, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 1580
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: [
                        $$contentTexts[7].value,
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            href: "https://dialog.react-component.vercel.app/",
                            children: $$contentTexts[8].value
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 22,
                            columnNumber: 1686
                        }, this)
                    ]
                }, void 0, true, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 1658
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "install",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#install",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/index.md",
                                lineNumber: 22,
                                columnNumber: 1841
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 22,
                            columnNumber: 1789
                        }, this),
                        "Install"
                    ]
                }, void 0, true, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 1772
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                        href: "https://npmjs.org/package/rc-dialog",
                        children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("img", {
                            src: "https://nodei.co/npm/rc-dialog.png",
                            alt: "rc-dialog"
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 22,
                            columnNumber: 1945
                        }, this)
                    }, void 0, false, {
                        fileName: "docs/index.md",
                        lineNumber: 22,
                        columnNumber: 1899
                    }, this)
                }, void 0, false, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 1896
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "usage",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#usage",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/index.md",
                                lineNumber: 22,
                                columnNumber: 2082
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 22,
                            columnNumber: 2032
                        }, this),
                        "Usage"
                    ]
                }, void 0, true, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 2017
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_index5.default, {
                    lang: "js",
                    children: $$contentTexts[9].value
                }, void 0, false, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 2135
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "api",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#api",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/index.md",
                                lineNumber: 22,
                                columnNumber: 2256
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 22,
                            columnNumber: 2208
                        }, this),
                        "API"
                    ]
                }, void 0, true, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 2195
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h3", {
                    id: "rc-dialog-1",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#rc-dialog-1",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/index.md",
                                lineNumber: 22,
                                columnNumber: 2384
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 22,
                            columnNumber: 2328
                        }, this),
                        "rc-dialog"
                    ]
                }, void 0, true, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 2307
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_index6.default, {
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("thead", {
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                children: [
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("th", {
                                        children: $$contentTexts[10].value
                                    }, void 0, false, {
                                        fileName: "docs/index.md",
                                        lineNumber: 22,
                                        columnNumber: 2459
                                    }, this),
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("th", {
                                        children: $$contentTexts[11].value
                                    }, void 0, false, {
                                        fileName: "docs/index.md",
                                        lineNumber: 22,
                                        columnNumber: 2494
                                    }, this),
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("th", {
                                        children: $$contentTexts[12].value
                                    }, void 0, false, {
                                        fileName: "docs/index.md",
                                        lineNumber: 22,
                                        columnNumber: 2529
                                    }, this),
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("th", {
                                        children: $$contentTexts[13].value
                                    }, void 0, false, {
                                        fileName: "docs/index.md",
                                        lineNumber: 22,
                                        columnNumber: 2564
                                    }, this),
                                    /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("th", {
                                        children: $$contentTexts[14].value
                                    }, void 0, false, {
                                        fileName: "docs/index.md",
                                        lineNumber: 22,
                                        columnNumber: 2599
                                    }, this)
                                ]
                            }, void 0, true, {
                                fileName: "docs/index.md",
                                lineNumber: 22,
                                columnNumber: 2455
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 22,
                            columnNumber: 2448
                        }, this),
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tbody", {
                            children: [
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[15].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 2658
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[16].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 2693
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[17].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 2728
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[18].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 2763
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 2798
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 2654
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[19].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 2813
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[20].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 2848
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 2883
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[21].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 2889
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 2924
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 2809
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[22].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 2939
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[23].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 2974
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3009
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[24].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3015
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3050
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 2935
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[25].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3065
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[26].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3100
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3135
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[27].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3141
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3176
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 3061
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[28].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3191
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[29].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3226
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[30].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3261
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[31].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3296
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3331
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 3187
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[32].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3346
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[33].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3381
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3416
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3422
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3428
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 3342
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[34].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3443
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[35].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3478
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[36].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3513
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[37].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3548
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3583
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 3439
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[38].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3598
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[39].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3633
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3668
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[40].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3674
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3709
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 3594
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[41].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3724
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[42].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3759
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3794
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[43].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3800
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3835
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 3720
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[44].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3850
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[45].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3885
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3920
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[46].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3926
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3961
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 3846
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[47].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 3976
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[48].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4011
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4046
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[49].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4052
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4087
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 3972
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[50].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4102
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[51].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4137
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4172
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[52].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4178
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4213
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 4098
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[53].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4228
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[54].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4263
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4298
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[55].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4304
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4339
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 4224
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[56].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4354
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[57].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4389
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[58].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4424
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[59].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4459
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4494
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 4350
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[60].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4509
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[61].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4544
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[62].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4579
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[63].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4614
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4649
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 4505
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[64].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4664
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[65].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4699
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[66].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4734
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[67].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4769
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4804
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 4660
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[68].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4819
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[69].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4854
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[70].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4889
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[71].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4924
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4959
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 4815
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[72].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 4974
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[73].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5009
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5044
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[74].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5050
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5085
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 4970
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[75].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5100
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[76].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5135
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5170
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[77].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5176
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5211
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 5096
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[78].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5226
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[79].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5261
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5296
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[80].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5302
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5337
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 5222
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[81].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5352
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[82].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5387
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5422
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[83].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5428
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5463
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 5348
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[84].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5478
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[85].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5513
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[86].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5548
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[87].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5583
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5618
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 5474
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[88].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5633
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[89].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5668
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5703
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[90].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5709
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5744
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 5629
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[91].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5759
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[92].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5794
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[93].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5829
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[94].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5864
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5899
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 5755
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[95].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5914
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[96].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5949
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[97].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 5984
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[98].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 6019
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 6054
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 5910
                                }, this),
                                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("tr", {
                                    children: [
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[99].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 6069
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[100].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 6104
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {}, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 6140
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[101].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 6146
                                        }, this),
                                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("td", {
                                            children: $$contentTexts[102].value
                                        }, void 0, false, {
                                            fileName: "docs/index.md",
                                            lineNumber: 22,
                                            columnNumber: 6182
                                        }, this)
                                    ]
                                }, void 0, true, {
                                    fileName: "docs/index.md",
                                    lineNumber: 22,
                                    columnNumber: 6065
                                }, this)
                            ]
                        }, void 0, true, {
                            fileName: "docs/index.md",
                            lineNumber: 22,
                            columnNumber: 2647
                        }, this)
                    ]
                }, void 0, true, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 2441
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "development",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#development",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/index.md",
                                lineNumber: 22,
                                columnNumber: 6316
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 22,
                            columnNumber: 6260
                        }, this),
                        "Development"
                    ]
                }, void 0, true, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 6239
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_index5.default, {
                    children: $$contentTexts[103].value
                }, void 0, false, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 6375
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "test-case",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#test-case",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/index.md",
                                lineNumber: 22,
                                columnNumber: 6500
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 22,
                            columnNumber: 6446
                        }, this),
                        "Test Case"
                    ]
                }, void 0, true, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 6427
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_index5.default, {
                    children: $$contentTexts[104].value
                }, void 0, false, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 6557
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "coverage",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#coverage",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/index.md",
                                lineNumber: 22,
                                columnNumber: 6680
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 22,
                            columnNumber: 6627
                        }, this),
                        "Coverage"
                    ]
                }, void 0, true, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 6609
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_index5.default, {
                    children: $$contentTexts[105].value
                }, void 0, false, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 6736
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: $$contentTexts[106].value
                }, void 0, false, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 6788
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("h2", {
                    id: "license",
                    children: [
                        /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("a", {
                            "aria-hidden": "true",
                            tabIndex: "-1",
                            href: "#license",
                            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("span", {
                                className: "icon icon-link"
                            }, void 0, false, {
                                fileName: "docs/index.md",
                                lineNumber: 22,
                                columnNumber: 6891
                            }, this)
                        }, void 0, false, {
                            fileName: "docs/index.md",
                            lineNumber: 22,
                            columnNumber: 6839
                        }, this),
                        "License"
                    ]
                }, void 0, true, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 6822
                }, this),
                /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)("p", {
                    children: $$contentTexts[107].value
                }, void 0, false, {
                    fileName: "docs/index.md",
                    lineNumber: 22,
                    columnNumber: 6946
                }, this)
            ]
        }, void 0, true, {
            fileName: "docs/index.md",
            lineNumber: 22,
            columnNumber: 12
        }, this)
    }, void 0, false);
}
_s(DumiMarkdownInner, "EsmAA1S45Kg2SXhnbajAzasObB8=", false, function() {
    return [
        _dumi.useRouteMeta
    ];
});
_c = DumiMarkdownInner;
// export named function for fastRefresh
// ref: https://github.com/pmmmwh/react-refresh-webpack-plugin/blob/main/docs/TROUBLESHOOTING.md#edits-always-lead-to-full-reload
function DumiMarkdownContent() {
    // wrap suspense for catch async meta data
    return /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_dumi.DumiPage, {
        children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_react.Suspense, {
            fallback: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(_loading.default, {}, void 0, false, {
                fileName: "docs/index.md",
                lineNumber: 29,
                columnNumber: 40
            }, void 0),
            children: /*#__PURE__*/ (0, _jsxdevruntime.jsxDEV)(DumiMarkdownInner, {}, void 0, false, {
                fileName: "docs/index.md",
                lineNumber: 29,
                columnNumber: 62
            }, this)
        }, void 0, false, {
            fileName: "docs/index.md",
            lineNumber: 29,
            columnNumber: 20
        }, this)
    }, void 0, false, {
        fileName: "docs/index.md",
        lineNumber: 29,
        columnNumber: 10
    }, this);
}
_c1 = DumiMarkdownContent;
var _default = DumiMarkdownContent;
var _c;
var _c1;
$RefreshReg$(_c, "DumiMarkdownInner");
$RefreshReg$(_c1, "DumiMarkdownContent");
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
 }]);
