// @rc-component/dialog/dist/meta__docs-async.js
((typeof globalThis !== 'undefined' ? globalThis : self)["makoChunk_@rc-component/dialog"] = (typeof globalThis !== 'undefined' ? globalThis : self)["makoChunk_@rc-component/dialog"] || []).push([
        ['meta__docs'],
{ "assets/bootstrap.less?asmodule": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
"";
var _default = {
    "rc-dialog-footer": `rc-dialog-footer-Ij5nLk7n`,
    "btn-group": `btn-group-lhTtdbkE`,
    "btn-block": `btn-block-_b4payKF`,
    "rc-dialog-fade-enter-active": `rc-dialog-fade-enter-active-PyQTED2R`,
    "rcDialogSlideFadeIn": `rcDialogSlideFadeIn-4lg_9oxA`,
    "btn": `btn-BW1TQvOA`,
    "rc-dialog-slide-fade-leave-active": `rc-dialog-slide-fade-leave-active-bXuJvxYQ`,
    "rcDialogFadeOut": `rcDialogFadeOut-O2CciMry`,
    "rc-dialog-mask-hidden": `rc-dialog-mask-hidden-KLrGIDRO`,
    "rcDialogFadeIn": `rcDialogFadeIn-DdxdDC_H`,
    "rc-dialog-slide-fade-appear-active": `rc-dialog-slide-fade-appear-active-kkhH3CYo`,
    "rc-dialog": `rc-dialog-xD_JwVZb`,
    "rc-dialog-fade-leave-active": `rc-dialog-fade-leave-active-wlJzTIzC`,
    "rc-dialog-slide-fade-enter": `rc-dialog-slide-fade-enter-r_15AXKS`,
    "rcDialogSlideFadeOut": `rcDialogSlideFadeOut-k2D9YUjG`,
    "rc-dialog-header": `rc-dialog-header-zh4gcC6c`,
    "rc-dialog-fade-leave": `rc-dialog-fade-leave-b2TVAZ3s`,
    "rc-dialog-fade-enter": `rc-dialog-fade-enter-a-gZsm73`,
    "rc-dialog-mask": `rc-dialog-mask-bD4P-Z0h`,
    "rc-dialog-close": `rc-dialog-close-EOweqtyx`,
    "rc-dialog-fade-appear-active": `rc-dialog-fade-appear-active-UABL8AdR`,
    "rc-dialog-slide-fade-appear": `rc-dialog-slide-fade-appear-4tK3vROB`,
    "rc-dialog-close-x": `rc-dialog-close-x-6kQX8jSl`,
    "rc-dialog-body": `rc-dialog-body-4XpdPXW4`,
    "rc-dialog-section": `rc-dialog-section-89lAKOoJ`,
    "rc-dialog-slide-fade-leave": `rc-dialog-slide-fade-leave-UJbeqi6K`,
    "rc-dialog-wrap": `rc-dialog-wrap-XaMOrmLh`,
    "rc-dialog-fade-appear": `rc-dialog-fade-appear--j1muc2F`,
    "rc-dialog-slide-fade-enter-active": `rc-dialog-slide-fade-enter-active-Bfj6xGTV`,
    "rc-dialog-title": `rc-dialog-title-56B7hstR`
};

},
"assets/bootstrap.less?dumi-raw": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
__mako_require__("assets/bootstrap.less?watch=parent");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _default = "@prefixCls: rc-dialog;\n@import \"./bootstrap/Dialog.less\";\n@import \"./index/Mask.less\";\n";
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"assets/bootstrap.less?watch=parent": function (module, exports, __mako_require__){

},
"assets/index.less?asmodule": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
"";
var _default = {
    "rc-dialog-footer": `rc-dialog-footer-9kI8P7T_`,
    "rc-dialog-fade-enter": `rc-dialog-fade-enter-baf6Ix3A`,
    "rc-dialog-fade-appear": `rc-dialog-fade-appear-HCuIGyzi`,
    "rc-dialog-fade-enter-active": `rc-dialog-fade-enter-active-bcJwE86b`,
    "rc-dialog-close": `rc-dialog-close-RpygIDJ7`,
    "rc-dialog-fade-appear-active": `rc-dialog-fade-appear-active-ImCuZCah`,
    "rc-dialog-zoom-appear": `rc-dialog-zoom-appear-O0oOmU02`,
    "rc-dialog-zoom-appear-active": `rc-dialog-zoom-appear-active-EksXhSiQ`,
    "rcDialogFadeOut": `rcDialogFadeOut-6hL4P9iD`,
    "rc-dialog-close-x": `rc-dialog-close-x-N1-X8nLa`,
    "rc-dialog-body": `rc-dialog-body-wophaOl0`,
    "rc-dialog-section": `rc-dialog-section-qyzOK45m`,
    "rcDialogZoomIn": `rcDialogZoomIn-qBju5MMS`,
    "rc-dialog-mask-hidden": `rc-dialog-mask-hidden-Q8OZyl0R`,
    "rc-dialog": `rc-dialog-s3ap7BnM`,
    "rc-dialog-wrap": `rc-dialog-wrap-lTGE5og0`,
    "rcDialogFadeIn": `rcDialogFadeIn-IjFW9mRF`,
    "rc-dialog-fade-leave-active": `rc-dialog-fade-leave-active-4C5kVyIf`,
    "rc-dialog-header": `rc-dialog-header-1r3oUdfe`,
    "rc-dialog-zoom-leave-active": `rc-dialog-zoom-leave-active-NCCySLyk`,
    "rc-dialog-fade-leave": `rc-dialog-fade-leave-lewBiqn6`,
    "rc-dialog-zoom-enter-active": `rc-dialog-zoom-enter-active-v7T3hJKP`,
    "rc-dialog-zoom-leave": `rc-dialog-zoom-leave-gtDKp35h`,
    "rc-dialog-zoom-enter": `rc-dialog-zoom-enter-U8H4By99`,
    "rcDialogZoomOut": `rcDialogZoomOut-cCRF_LGx`,
    "rc-dialog-mask": `rc-dialog-mask-Xf46-IWW`,
    "rc-dialog-title": `rc-dialog-title-sZg4rzBr`
};

},
"assets/index.less?dumi-raw": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
__mako_require__("assets/index.less?watch=parent");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _default = "@prefixCls: rc-dialog;\n\n@import \"./index/Dialog.less\";\n@import \"./index/Mask.less\";\n";
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"assets/index.less?watch=parent": function (module, exports, __mako_require__){

},
"docs/changelog.md?type=demo": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "demos", {
    enumerable: true,
    get: function() {
        return demos;
    }
});
var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
var _react = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_react@18.3.1@react/index.js"));
__mako_require__("docs/changelog.md?watch=parent");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const demos = {};
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/changelog.md?type=text": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "texts", {
    enumerable: true,
    get: function() {
        return texts;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
__mako_require__("docs/changelog.md?watch=parent");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const texts = [
    {
        "value": "2021-01-07",
        "paraId": 0,
        "tocIndex": 1
    },
    {
        "value": "improve: ScrollLocker call related. ",
        "paraId": 1,
        "tocIndex": 1
    },
    {
        "value": "#227",
        "paraId": 1,
        "tocIndex": 1
    },
    {
        "value": "2021-01-04",
        "paraId": 2,
        "tocIndex": 2
    },
    {
        "value": "refactor: use ScrollLocker. ",
        "paraId": 3,
        "tocIndex": 2
    },
    {
        "value": "#220",
        "paraId": 3,
        "tocIndex": 2
    },
    {
        "value": "2020-12-19",
        "paraId": 4,
        "tocIndex": 3
    },
    {
        "value": "fix: trigger close only when click wrap itself. ",
        "paraId": 5,
        "tocIndex": 3
    },
    {
        "value": "#218",
        "paraId": 5,
        "tocIndex": 3
    },
    {
        "value": "2020-12-07",
        "paraId": 6,
        "tocIndex": 4
    },
    {
        "value": "fix: Dialog should keep refresh when forceRender provided. ",
        "paraId": 7,
        "tocIndex": 4
    },
    {
        "value": "#217",
        "paraId": 7,
        "tocIndex": 4
    },
    {
        "value": "2020-12-03",
        "paraId": 8,
        "tocIndex": 5
    },
    {
        "value": "fix: dialog dont close when mouseDown in content and mouseUp in wrapper. ",
        "paraId": 9,
        "tocIndex": 5
    },
    {
        "value": "#210",
        "paraId": 9,
        "tocIndex": 5
    },
    {
        "value": "chore: Should not re-render when dialog is invisible. ",
        "paraId": 10,
        "tocIndex": 5
    },
    {
        "value": "#212",
        "paraId": 10,
        "tocIndex": 5
    },
    {
        "value": "2020-10-21",
        "paraId": 11,
        "tocIndex": 6
    },
    {
        "value": "chore: support react 17. ",
        "paraId": 12,
        "tocIndex": 6
    },
    {
        "value": "#207",
        "paraId": 12,
        "tocIndex": 6
    },
    {
        "value": "2020-10-14",
        "paraId": 13,
        "tocIndex": 7
    },
    {
        "value": "fix: Dialog should not auto destroy. ",
        "paraId": 14,
        "tocIndex": 7
    },
    {
        "value": "#206",
        "paraId": 14,
        "tocIndex": 7
    },
    {
        "value": "2020-10-11",
        "paraId": 15,
        "tocIndex": 8
    },
    {
        "value": "fix: Portal event bubble. ",
        "paraId": 16,
        "tocIndex": 8
    },
    {
        "value": "#204",
        "paraId": 16,
        "tocIndex": 8
    },
    {
        "value": "2020-09-29",
        "paraId": 17,
        "tocIndex": 9
    },
    {
        "value": "refactor: Use ",
        "paraId": 18,
        "tocIndex": 9
    },
    {
        "value": "rc-motion",
        "paraId": 18,
        "tocIndex": 9
    },
    {
        "value": ". ",
        "paraId": 18,
        "tocIndex": 9
    },
    {
        "value": "#203",
        "paraId": 18,
        "tocIndex": 9
    },
    {
        "value": "fix: prevent scroll behavior when focus trigger. ",
        "paraId": 19,
        "tocIndex": 10
    },
    {
        "value": "ant-design/ant-design#26582",
        "paraId": 19,
        "tocIndex": 10
    },
    {
        "value": "fix: page scroll position will jump after dialog is closed. ",
        "paraId": 20,
        "tocIndex": 11
    },
    {
        "value": "#202",
        "paraId": 20,
        "tocIndex": 11
    },
    {
        "value": "fix: remove typing from package.json. ",
        "paraId": 21,
        "tocIndex": 12
    },
    {
        "value": "#201",
        "paraId": 21,
        "tocIndex": 12
    },
    {
        "value": "fix: add displayName. ",
        "paraId": 22,
        "tocIndex": 13
    },
    {
        "value": "#200",
        "paraId": 22,
        "tocIndex": 13
    },
    {
        "value": "fate: add ",
        "paraId": 23,
        "tocIndex": 14
    },
    {
        "value": "modalRender",
        "paraId": 23,
        "tocIndex": 14
    },
    {
        "value": ". ",
        "paraId": 23,
        "tocIndex": 14
    },
    {
        "value": "#195",
        "paraId": 23,
        "tocIndex": 14
    },
    {
        "value": "use ",
        "paraId": 24,
        "tocIndex": 15
    },
    {
        "value": "father",
        "paraId": 24,
        "tocIndex": 15
    },
    {
        "value": ". ",
        "paraId": 24,
        "tocIndex": 15
    },
    {
        "value": "#197",
        "paraId": 24,
        "tocIndex": 15
    },
    {
        "value": "Fix dialog component will only show mask, if initialize a Dialog component with both forceRender and visible are true. ",
        "paraId": 25,
        "tocIndex": 16
    },
    {
        "value": "#194",
        "paraId": 25,
        "tocIndex": 16
    },
    {
        "value": "remove babel runtime.",
        "paraId": 26,
        "tocIndex": 17
    },
    {
        "value": "up ",
        "paraId": 26,
        "tocIndex": 17
    },
    {
        "value": "rc-drawer",
        "paraId": 26,
        "tocIndex": 17
    },
    {
        "value": " to ",
        "paraId": 26,
        "tocIndex": 17
    },
    {
        "value": "4.1.0",
        "paraId": 26,
        "tocIndex": 17
    },
    {
        "value": ".",
        "paraId": 26,
        "tocIndex": 17
    },
    {
        "value": "upgrade ",
        "paraId": 27,
        "tocIndex": 18
    },
    {
        "value": "rc-util",
        "paraId": 27,
        "tocIndex": 18
    },
    {
        "value": " to ",
        "paraId": 27,
        "tocIndex": 18
    },
    {
        "value": "5.0.0",
        "paraId": 27,
        "tocIndex": 18
    },
    {
        "value": ".",
        "paraId": 27,
        "tocIndex": 18
    },
    {
        "value": "upgrade ",
        "paraId": 28,
        "tocIndex": 19
    },
    {
        "value": "rc-animate",
        "paraId": 28,
        "tocIndex": 19
    },
    {
        "value": " to ",
        "paraId": 28,
        "tocIndex": 19
    },
    {
        "value": "3.0.0",
        "paraId": 28,
        "tocIndex": 19
    },
    {
        "value": ".",
        "paraId": 28,
        "tocIndex": 19
    },
    {
        "value": "Update accessibility.",
        "paraId": 29,
        "tocIndex": 20
    },
    {
        "value": "Support ",
        "paraId": 30,
        "tocIndex": 21
    },
    {
        "value": "forceRender",
        "paraId": 30,
        "tocIndex": 21
    },
    {
        "value": " for dialog.",
        "paraId": 30,
        "tocIndex": 21
    },
    {
        "value": "Add closeIcon. ",
        "paraId": 31,
        "tocIndex": 22
    },
    {
        "value": "#89",
        "paraId": 32,
        "tocIndex": 22
    },
    {
        "value": " ",
        "paraId": 31,
        "tocIndex": 22
    },
    {
        "value": "@HeskeyBaozi ",
        "paraId": 31,
        "tocIndex": 22
    },
    {
        "value": "Add destroyOnClose. ",
        "paraId": 33,
        "tocIndex": 23
    },
    {
        "value": "#72",
        "paraId": 33,
        "tocIndex": 23
    },
    {
        "value": " ",
        "paraId": 33,
        "tocIndex": 23
    },
    {
        "value": "@Rohanhacker",
        "paraId": 33,
        "tocIndex": 23
    },
    {
        "value": "Remove ReactNative support, please use ",
        "paraId": 34,
        "tocIndex": 24
    },
    {
        "value": "https://github.com/react-component/m-dialog",
        "paraId": 34,
        "tocIndex": 24
    },
    {
        "value": " instead.",
        "paraId": 34,
        "tocIndex": 24
    },
    {
        "value": "Support React 16.",
        "paraId": 34,
        "tocIndex": 24
    },
    {
        "value": "Notable change: Close animation won't trigger when dialog unmounting after React 16, see ",
        "paraId": 35,
        "tocIndex": 24
    },
    {
        "value": "facebook/react#10826",
        "paraId": 35,
        "tocIndex": 24
    },
    {
        "value": "fixed: RN modal support landscape orientation, ",
        "paraId": 36,
        "tocIndex": 25
    },
    {
        "value": "https://github.com/react-component/dialog/pull/64",
        "paraId": 36,
        "tocIndex": 25
    },
    {
        "value": "remove rc-dialog/lib/Modal's entry prop, add animationType prop",
        "paraId": 37,
        "tocIndex": 26
    },
    {
        "value": "add rc-dialog/lib/Modal to support react-native",
        "paraId": 38,
        "tocIndex": 27
    },
    {
        "value": "use getContainerRenderMixin from 'rc-util'",
        "paraId": 39,
        "tocIndex": 28
    },
    {
        "value": "new html structure and class",
        "paraId": 40,
        "tocIndex": 29
    },
    {
        "value": "disable window scroll when show",
        "paraId": 40,
        "tocIndex": 29
    },
    {
        "value": "add maskClosable",
        "paraId": 41,
        "tocIndex": 30
    },
    {
        "value": "separate close and header",
        "paraId": 42,
        "tocIndex": 31
    },
    {
        "value": "only support react 0.14",
        "paraId": 43,
        "tocIndex": 32
    },
    {
        "value": "refactor to clean api. remove onShow onBeforeClose",
        "paraId": 44,
        "tocIndex": 33
    },
    {
        "value": "use rc-animate & rc-align",
        "paraId": 45,
        "tocIndex": 34
    },
    {
        "value": "support esc to close",
        "paraId": 46,
        "tocIndex": 35
    },
    {
        "value": "add renderToBody props",
        "paraId": 47,
        "tocIndex": 36
    },
    {
        "value": "make dialog render to body and use ",
        "paraId": 48,
        "tocIndex": 37
    },
    {
        "value": "dom-align",
        "paraId": 48,
        "tocIndex": 37
    },
    {
        "value": " to align",
        "paraId": 48,
        "tocIndex": 37
    },
    {
        "value": "support es6 and react 0.13",
        "paraId": 49,
        "tocIndex": 38
    },
    {
        "value": "new",
        "paraId": 50,
        "tocIndex": 39
    },
    {
        "value": " ",
        "paraId": 50,
        "tocIndex": 39
    },
    {
        "value": "#3",
        "paraId": 50,
        "tocIndex": 39
    },
    {
        "value": " support closable requestClose onBeforeClose",
        "paraId": 50,
        "tocIndex": 39
    }
];
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/demo/ant-design.md?type=demo": function (module, exports, __mako_require__){
"use strict";
var interop = __mako_require__("@swc/helpers/_/_interop_require_wildcard")._;
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "demos", {
    enumerable: true,
    get: function() {
        return demos;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react@18.3.1@react/index.js"));
__mako_require__("docs/demo/ant-design.md?watch=parent");
var _rcselect = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_rc-select@14.16.6@rc-select/es/index.js"));
var _indexlessasmodule = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_rc-select@14.16.6@rc-select/assets/index.less?asmodule"));
var _rcdialog = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("src/index.ts"));
var _indexlessasmodule1 = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("assets/index.less?asmodule"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const demos = {
    'docs-demo-ant-design-demo-ant-design': {
        component: /*#__PURE__*/ _react.default.memo(/*#__PURE__*/ _react.default.lazy(()=>__mako_require__.ensure2("docs/examples/ant-design.tsx?techStack=react").then(__mako_require__.dr(interop, __mako_require__.bind(__mako_require__, "docs/examples/ant-design.tsx?techStack=react"))))),
        asset: {
            "type": "BLOCK",
            "id": "docs-demo-ant-design-demo-ant-design",
            "refAtomIds": [],
            "dependencies": {
                "index.tsx": {
                    "type": "FILE",
                    "value": __mako_require__("docs/examples/ant-design.tsx?dumi-raw").default
                },
                "react": {
                    "type": "NPM",
                    "value": "18.3.1"
                },
                "rc-select": {
                    "type": "NPM",
                    "value": "14.16.6"
                },
                "@rc-component/dialog": {
                    "type": "NPM",
                    "value": "1.0.1"
                },
                "../../assets/index.less": {
                    "type": "FILE",
                    "value": __mako_require__("assets/index.less?dumi-raw").default
                }
            },
            "entry": "index.tsx"
        },
        context: {
            "../../assets/index.less": _indexlessasmodule1,
            "react": _react,
            "rc-select": _rcselect,
            "rc-select/assets/index.less": _indexlessasmodule,
            "rc-dialog": _rcdialog,
            "/Users/jilin/projects/antd/rc-dialog/assets/index.less": _indexlessasmodule1
        },
        renderOpts: {
            compile: async (...args)=>{
                return (await __mako_require__.ensure2("node_modules/_dumi@2.4.17@dumi/dist/client/misc/reactDemoCompiler.js").then(__mako_require__.dr(interop, __mako_require__.bind(__mako_require__, "node_modules/_dumi@2.4.17@dumi/dist/client/misc/reactDemoCompiler.js")))).default(...args);
            }
        }
    }
};
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/demo/ant-design.md?type=text": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "texts", {
    enumerable: true,
    get: function() {
        return texts;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
__mako_require__("docs/demo/ant-design.md?watch=parent");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const texts = [];
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/demo/bootstrap.md?type=demo": function (module, exports, __mako_require__){
"use strict";
var interop = __mako_require__("@swc/helpers/_/_interop_require_wildcard")._;
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "demos", {
    enumerable: true,
    get: function() {
        return demos;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react@18.3.1@react/index.js"));
__mako_require__("docs/demo/bootstrap.md?watch=parent");
var _rcdialog = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("src/index.ts"));
var _bootstrapcssasmodule = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_bootstrap@4.6.2@bootstrap/dist/css/bootstrap.css?asmodule"));
var _bootstraplessasmodule = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("assets/bootstrap.less?asmodule"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const demos = {
    'docs-demo-bootstrap-demo-bootstrap': {
        component: /*#__PURE__*/ _react.default.memo(/*#__PURE__*/ _react.default.lazy(()=>__mako_require__.ensure2("docs/examples/bootstrap.tsx?techStack=react").then(__mako_require__.dr(interop, __mako_require__.bind(__mako_require__, "docs/examples/bootstrap.tsx?techStack=react"))))),
        asset: {
            "type": "BLOCK",
            "id": "docs-demo-bootstrap-demo-bootstrap",
            "refAtomIds": [],
            "dependencies": {
                "index.tsx": {
                    "type": "FILE",
                    "value": __mako_require__("docs/examples/bootstrap.tsx?dumi-raw").default
                },
                "react": {
                    "type": "NPM",
                    "value": "18.3.1"
                },
                "@rc-component/dialog": {
                    "type": "NPM",
                    "value": "1.0.1"
                },
                "bootstrap": {
                    "type": "NPM",
                    "value": "4.6.2"
                },
                "../../assets/bootstrap.less": {
                    "type": "FILE",
                    "value": __mako_require__("assets/bootstrap.less?dumi-raw").default
                }
            },
            "entry": "index.tsx"
        },
        context: {
            "../../assets/bootstrap.less": _bootstraplessasmodule,
            "react": _react,
            "rc-dialog": _rcdialog,
            "bootstrap/dist/css/bootstrap.css": _bootstrapcssasmodule,
            "/Users/jilin/projects/antd/rc-dialog/assets/bootstrap.less": _bootstraplessasmodule
        },
        renderOpts: {
            compile: async (...args)=>{
                return (await __mako_require__.ensure2("node_modules/_dumi@2.4.17@dumi/dist/client/misc/reactDemoCompiler.js").then(__mako_require__.dr(interop, __mako_require__.bind(__mako_require__, "node_modules/_dumi@2.4.17@dumi/dist/client/misc/reactDemoCompiler.js")))).default(...args);
            }
        }
    }
};
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/demo/bootstrap.md?type=text": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "texts", {
    enumerable: true,
    get: function() {
        return texts;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
__mako_require__("docs/demo/bootstrap.md?watch=parent");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const texts = [];
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/demo/draggable.md?type=demo": function (module, exports, __mako_require__){
"use strict";
var interop = __mako_require__("@swc/helpers/_/_interop_require_wildcard")._;
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "demos", {
    enumerable: true,
    get: function() {
        return demos;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react@18.3.1@react/index.js"));
__mako_require__("docs/demo/draggable.md?watch=parent");
var _bootstrapcssasmodule = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_bootstrap@4.6.2@bootstrap/dist/css/bootstrap.css?asmodule"));
var _reactdraggable = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-draggable@4.4.6@react-draggable/build/cjs/cjs.js"));
var _rcdialog = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("src/index.ts"));
var _indexlessasmodule = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("assets/index.less?asmodule"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const demos = {
    'docs-demo-draggable-demo-draggable': {
        component: /*#__PURE__*/ _react.default.memo(/*#__PURE__*/ _react.default.lazy(()=>__mako_require__.ensure2("docs/examples/draggable.tsx?techStack=react").then(__mako_require__.dr(interop, __mako_require__.bind(__mako_require__, "docs/examples/draggable.tsx?techStack=react"))))),
        asset: {
            "type": "BLOCK",
            "id": "docs-demo-draggable-demo-draggable",
            "refAtomIds": [],
            "dependencies": {
                "index.tsx": {
                    "type": "FILE",
                    "value": __mako_require__("docs/examples/draggable.tsx?dumi-raw").default
                },
                "bootstrap": {
                    "type": "NPM",
                    "value": "4.6.2"
                },
                "react": {
                    "type": "NPM",
                    "value": "18.3.1"
                },
                "react-draggable": {
                    "type": "NPM",
                    "value": "4.4.6"
                },
                "@rc-component/dialog": {
                    "type": "NPM",
                    "value": "1.0.1"
                },
                "../../assets/index.less": {
                    "type": "FILE",
                    "value": __mako_require__("assets/index.less?dumi-raw").default
                }
            },
            "entry": "index.tsx"
        },
        context: {
            "../../assets/index.less": _indexlessasmodule,
            "bootstrap/dist/css/bootstrap.css": _bootstrapcssasmodule,
            "react": _react,
            "react-draggable": _reactdraggable,
            "rc-dialog": _rcdialog,
            "/Users/jilin/projects/antd/rc-dialog/assets/index.less": _indexlessasmodule
        },
        renderOpts: {
            compile: async (...args)=>{
                return (await __mako_require__.ensure2("node_modules/_dumi@2.4.17@dumi/dist/client/misc/reactDemoCompiler.js").then(__mako_require__.dr(interop, __mako_require__.bind(__mako_require__, "node_modules/_dumi@2.4.17@dumi/dist/client/misc/reactDemoCompiler.js")))).default(...args);
            }
        }
    }
};
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/demo/draggable.md?type=text": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "texts", {
    enumerable: true,
    get: function() {
        return texts;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
__mako_require__("docs/demo/draggable.md?watch=parent");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const texts = [];
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/demo/multiple-Portal.md?type=demo": function (module, exports, __mako_require__){
"use strict";
var interop = __mako_require__("@swc/helpers/_/_interop_require_wildcard")._;
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "demos", {
    enumerable: true,
    get: function() {
        return demos;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react@18.3.1@react/index.js"));
__mako_require__("docs/demo/multiple-Portal.md?watch=parent");
var _rcdrawer = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_rc-drawer@7.2.0@rc-drawer/es/index.js"));
var _indexcssasmodule = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_rc-drawer@7.2.0@rc-drawer/assets/index.css?asmodule"));
var _rcdialog = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("src/index.ts"));
var _indexlessasmodule = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("assets/index.less?asmodule"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const demos = {
    'docs-demo-multiple-portal-demo-multiple-portal': {
        component: /*#__PURE__*/ _react.default.memo(/*#__PURE__*/ _react.default.lazy(()=>__mako_require__.ensure2("docs/examples/multiple-Portal.tsx?techStack=react").then(__mako_require__.dr(interop, __mako_require__.bind(__mako_require__, "docs/examples/multiple-Portal.tsx?techStack=react"))))),
        asset: {
            "type": "BLOCK",
            "id": "docs-demo-multiple-portal-demo-multiple-portal",
            "refAtomIds": [],
            "dependencies": {
                "index.tsx": {
                    "type": "FILE",
                    "value": __mako_require__("docs/examples/multiple-Portal.tsx?dumi-raw").default
                },
                "react": {
                    "type": "NPM",
                    "value": "18.3.1"
                },
                "rc-drawer": {
                    "type": "NPM",
                    "value": "7.2.0"
                },
                "@rc-component/dialog": {
                    "type": "NPM",
                    "value": "1.0.1"
                },
                "../../assets/index.less": {
                    "type": "FILE",
                    "value": __mako_require__("assets/index.less?dumi-raw").default
                }
            },
            "entry": "index.tsx"
        },
        context: {
            "../../assets/index.less": _indexlessasmodule,
            "react": _react,
            "rc-drawer": _rcdrawer,
            "rc-drawer/assets/index.css": _indexcssasmodule,
            "rc-dialog": _rcdialog,
            "/Users/jilin/projects/antd/rc-dialog/assets/index.less": _indexlessasmodule
        },
        renderOpts: {
            compile: async (...args)=>{
                return (await __mako_require__.ensure2("node_modules/_dumi@2.4.17@dumi/dist/client/misc/reactDemoCompiler.js").then(__mako_require__.dr(interop, __mako_require__.bind(__mako_require__, "node_modules/_dumi@2.4.17@dumi/dist/client/misc/reactDemoCompiler.js")))).default(...args);
            }
        }
    }
};
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/demo/multiple-Portal.md?type=text": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "texts", {
    enumerable: true,
    get: function() {
        return texts;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
__mako_require__("docs/demo/multiple-Portal.md?watch=parent");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const texts = [];
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/demo/pure.md?type=demo": function (module, exports, __mako_require__){
"use strict";
var interop = __mako_require__("@swc/helpers/_/_interop_require_wildcard")._;
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "demos", {
    enumerable: true,
    get: function() {
        return demos;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
var _react = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react@18.3.1@react/index.js"));
__mako_require__("docs/demo/pure.md?watch=parent");
var _rcdialog = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("src/index.ts"));
var _indexlessasmodule = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_rc-select@14.16.6@rc-select/assets/index.less?asmodule"));
var _indexlessasmodule1 = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("assets/index.less?asmodule"));
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const demos = {
    'docs-demo-pure-demo-pure': {
        component: /*#__PURE__*/ _react.default.memo(/*#__PURE__*/ _react.default.lazy(()=>__mako_require__.ensure2("docs/examples/pure.tsx?techStack=react").then(__mako_require__.dr(interop, __mako_require__.bind(__mako_require__, "docs/examples/pure.tsx?techStack=react"))))),
        asset: {
            "type": "BLOCK",
            "id": "docs-demo-pure-demo-pure",
            "refAtomIds": [],
            "dependencies": {
                "index.tsx": {
                    "type": "FILE",
                    "value": __mako_require__("docs/examples/pure.tsx?dumi-raw").default
                },
                "react": {
                    "type": "NPM",
                    "value": "18.3.1"
                },
                "@rc-component/dialog": {
                    "type": "NPM",
                    "value": "1.0.1"
                },
                "rc-select": {
                    "type": "NPM",
                    "value": "14.16.6"
                },
                "../../assets/index.less": {
                    "type": "FILE",
                    "value": __mako_require__("assets/index.less?dumi-raw").default
                }
            },
            "entry": "index.tsx"
        },
        context: {
            "../../assets/index.less": _indexlessasmodule1,
            "react": _react,
            "rc-dialog": _rcdialog,
            "rc-select/assets/index.less": _indexlessasmodule,
            "/Users/jilin/projects/antd/rc-dialog/assets/index.less": _indexlessasmodule1
        },
        renderOpts: {
            compile: async (...args)=>{
                return (await __mako_require__.ensure2("node_modules/_dumi@2.4.17@dumi/dist/client/misc/reactDemoCompiler.js").then(__mako_require__.dr(interop, __mako_require__.bind(__mako_require__, "node_modules/_dumi@2.4.17@dumi/dist/client/misc/reactDemoCompiler.js")))).default(...args);
            }
        }
    }
};
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/demo/pure.md?type=text": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "texts", {
    enumerable: true,
    get: function() {
        return texts;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
__mako_require__("docs/demo/pure.md?watch=parent");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const texts = [];
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/examples/ant-design.tsx?dumi-raw": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
__mako_require__("docs/examples/ant-design.tsx?watch=parent");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _default = "/* eslint no-console:0 */\nimport * as React from 'react';\nimport Select from 'rc-select';\nimport 'rc-select/assets/index.less';\nimport Dialog from 'rc-dialog';\nimport '../../assets/index.less';\n\nconst clearPath =\n  'M793 242H366v-74c0-6.7-7.7-10.4-12.9' +\n  '-6.3l-142 112c-4.1 3.2-4.1 9.4 0 12.6l142 112c' +\n  '5.2 4.1 12.9 0.4 12.9-6.3v-74h415v470H175c-4.4' +\n  ' 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h618c35.3 0 64-' +\n  '28.7 64-64V306c0-35.3-28.7-64-64-64z';\n\nconst getSvg = (path: string, props = {}, align = false) => (\n  <i {...props}>\n    <svg\n      viewBox=\"0 0 1024 1024\"\n      width=\"1em\"\n      height=\"1em\"\n      fill=\"currentColor\"\n      style={align ? { verticalAlign: '-0.125em ' } : {}}\n    >\n      <path d={path} />\n    </svg>\n  </i>\n);\n\nconst MyControl: React.FC = () => {\n  const [visible1, setVisible1] = React.useState(true);\n  const [visible2, setVisible2] = React.useState(false);\n  const [visible3, setVisible3] = React.useState(false);\n  const [width, setWidth] = React.useState(600);\n  const [destroyOnClose, setDestroyOnClose] = React.useState(false);\n  const [center, setCenter] = React.useState(false);\n  const [mousePosition, setMousePosition] = React.useState({ x: null, y: null });\n  const [useIcon, setUseIcon] = React.useState(false);\n  const [forceRender, setForceRender] = React.useState(false);\n\n  const onClick = (e: React.MouseEvent) => {\n    setMousePosition({ x: e.pageX, y: e.pageY });\n    setVisible1(true);\n  };\n\n  const onClose = () => {\n    setVisible1(false);\n  };\n\n  const onClose2 = () => {\n    setVisible2(false);\n  };\n\n  const onClose3 = () => {\n    setVisible3(false);\n  };\n\n  const closeAll = () => {\n    setVisible1(false);\n    setVisible2(false);\n    setVisible3(false);\n  };\n\n  const onDestroyOnCloseChange = (e: React.ChangeEvent<HTMLInputElement>) => {\n    setDestroyOnClose(e.target.checked);\n  };\n\n  const onForceRenderChange = (e: React.ChangeEvent<HTMLInputElement>) => {\n    setForceRender(e.target.checked);\n  };\n\n  const changeWidth = () => {\n    setWidth(width === 600 ? 800 : 600);\n  };\n\n  const centerEvent = (e: React.ChangeEvent<HTMLInputElement>) => {\n    setCenter(e.target.checked);\n  };\n\n  const toggleCloseIcon = () => {\n    setUseIcon(!useIcon);\n  };\n\n  const style = { width };\n\n  let wrapClassName = '';\n  if (center) {\n    wrapClassName = 'center';\n  }\n\n  const dialog = (\n    <Dialog\n      visible={visible1}\n      wrapClassName={wrapClassName}\n      animation=\"zoom\"\n      maskAnimation=\"fade\"\n      onClose={onClose}\n      style={style}\n      title=\"dialog1\"\n      mousePosition={mousePosition}\n      destroyOnClose={destroyOnClose}\n      closeIcon={useIcon ? getSvg(clearPath, {}, true) : undefined}\n      forceRender={forceRender}\n      focusTriggerAfterClose={false}\n    >\n      <input autoFocus />\n      <p>basic modal</p>\n      <button\n        type=\"button\"\n        onClick={() => {\n          setVisible1(false);\n          setVisible2(true);\n        }}\n      >\n        \n      </button>\n      <button\n        type=\"button\"\n        onClick={() => {\n          setVisible2(true);\n        }}\n      >\n        \n      </button>\n      <button type=\"button\" onClick={changeWidth}>\n        change width\n      </button>\n      <button type=\"button\" onClick={toggleCloseIcon}>\n        use custom icon, is using icon: {(useIcon && 'true') || 'false'}.\n      </button>\n      <div style={{ height: 200 }}>\n        <Select dropdownStyle={{ zIndex: 9999999 }}>\n          <Select.Option value=\"light\">Light</Select.Option>\n        </Select>\n      </div>\n    </Dialog>\n  );\n\n  const dialog2 = (\n    <Dialog visible={visible2} title=\"dialog2\" onClose={onClose2}>\n      <input autoFocus />\n      <p>basic modal</p>\n      <button\n        type=\"button\"\n        onClick={() => {\n          setVisible3(true);\n        }}\n      >\n        \n      </button>\n      <button\n        type=\"button\"\n        onClick={() => {\n          setVisible2(false);\n        }}\n      >\n        \n      </button>\n      <button type=\"button\" onClick={closeAll}>\n        \n      </button>\n      <button type=\"button\" onClick={changeWidth}>\n        change width\n      </button>\n      <button type=\"button\" onClick={toggleCloseIcon}>\n        use custom icon, is using icon: {(useIcon && 'true') || 'false'}.\n      </button>\n      <div style={{ height: 200 }} />\n    </Dialog>\n  );\n\n  const dialog3 = (\n    <Dialog forceRender title=\"dialog3\" visible={visible3} onClose={onClose3}>\n      <p>initialized with forceRender and visbile true</p>\n      <button\n        type=\"button\"\n        onClick={() => {\n          setVisible3(false);\n        }}\n      >\n        \n      </button>\n      <button type=\"button\" onClick={closeAll}>\n        \n      </button>\n      <button type=\"button\" onClick={changeWidth}>\n        change width\n      </button>\n      <button type=\"button\" onClick={toggleCloseIcon}>\n        use custom icon, is using icon: {(useIcon && 'true') || 'false'}.\n      </button>\n      <div style={{ height: 200 }} />\n    </Dialog>\n  );\n\n  return (\n    <React.StrictMode>\n      <div style={{ width: '90%', margin: '0 auto', height: '150vh' }}>\n        <style>\n          {`\n          .center {\n            display: flex;\n            align-items: center;\n            justify-content: center;\n          }\n          `}\n        </style>\n        <p>\n          <button type=\"button\" className=\"btn btn-primary\" onClick={onClick}>\n            show dialog\n          </button>\n          &nbsp;\n          <label>\n            destroy on close:\n            <input type=\"checkbox\" checked={destroyOnClose} onChange={onDestroyOnCloseChange} />\n          </label>\n          &nbsp;\n          <label>\n            center\n            <input type=\"checkbox\" checked={center} onChange={centerEvent} />\n          </label>\n          &nbsp;\n          <label>\n            force render\n            <input type=\"checkbox\" checked={forceRender} onChange={onForceRenderChange} />\n          </label>\n          <input placeholder=\"Useless Input\" onClick={onClick} />\n        </p>\n        {dialog}\n        {dialog2}\n        {dialog3}\n      </div>\n    </React.StrictMode>\n  );\n};\n\nexport default MyControl;\n";
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/examples/bootstrap.tsx?dumi-raw": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
__mako_require__("docs/examples/bootstrap.tsx?watch=parent");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _default = "import React from 'react';\nimport Dialog from 'rc-dialog';\nimport 'bootstrap/dist/css/bootstrap.css';\nimport '../../assets/bootstrap.less';\n\n// Check for memo update should work\nconst InnerRender: React.FC = () => {\n  console.log('Updated...', Date.now());\n  return null;\n};\n\nconst MyControl: React.FC = () => {\n  const [visible, setVisible] = React.useState(false);\n  const [destroyOnClose, setDestroyOnClose] = React.useState(false);\n\n  const onClick = () => {\n    setVisible(true);\n  };\n\n  const onClose = () => {\n    setVisible(false);\n  };\n\n  const onDestroyOnCloseChange = (e: React.ChangeEvent<HTMLInputElement>) => {\n    setDestroyOnClose(e.target.checked);\n  };\n\n  const dialog = (\n    <Dialog\n      visible={visible}\n      destroyOnClose={destroyOnClose}\n      animation=\"slide-fade\"\n      maskAnimation=\"fade\"\n      onClose={onClose}\n      style={{ width: 600 }}\n      title={<div></div>}\n      footer={[\n        <button type=\"button\" className=\"btn btn-default\" key=\"close\" onClick={onClose}>\n          Close\n        </button>,\n        <button type=\"button\" className=\"btn btn-primary\" key=\"save\" onClick={onClose}>\n          Save changes\n        </button>,\n      ]}\n    >\n      <InnerRender />\n      <h4>Text in a modal</h4>\n      <p>Duis mollis, est non commodo luctus, nisi erat porttitor ligula.</p>\n      <hr />\n      <h4>Overflowing text to show scroll behavior</h4>\n      <p>\n        Cras mattis consectetur purus sit amet fermentum. Cras justo odio, dapibus ac facilisis in,\n        egestas eget quam. Morbi leo risus, porta ac consectetur ac, vestibulum at eros.\n      </p>\n      <p>\n        <button type=\"button\" className=\"btn btn-primary\">\n          Primary\n        </button>{' '}\n        <button type=\"button\" className=\"btn btn-secondary\">\n          Secondary\n        </button>{' '}\n        <button type=\"button\" className=\"btn btn-success\">\n          Success\n        </button>{' '}\n        <button type=\"button\" className=\"btn btn-danger\">\n          Danger\n        </button>{' '}\n        <button type=\"button\" className=\"btn btn-warning\">\n          Warning\n        </button>{' '}\n        <button type=\"button\" className=\"btn btn-info\">\n          Info\n        </button>{' '}\n      </p>\n      <p>\n        Praesent commodo cursus magna, vel scelerisque nisl consectetur et. Vivamus sagittis lacus\n        vel augue laoreet rutrum faucibus dolor auctor.\n      </p>\n      <div style={{ display: '' }}>\n        <p>\n          Aenean lacinia bibendum nulla sed consectetur. Praesent commodo cursus magna, vel\n          scelerisque nisl consectetur et. Donec sed odio dui. Donec ullamcorper nulla non metus\n          auctor fringilla.\n        </p>\n        <p>\n          Cras mattis consectetur purus sit amet fermentum. Cras justo odio, dapibus ac facilisis\n          in, egestas eget quam. Morbi leo risus, porta ac consectetur ac, vestibulum at eros.\n        </p>\n        <p>\n          Praesent commodo cursus magna, vel scelerisque nisl consectetur et. Vivamus sagittis lacus\n          vel augue laoreet rutrum faucibus dolor auctor.\n        </p>\n        <p>\n          Aenean lacinia bibendum nulla sed consectetur. Praesent commodo cursus magna, vel\n          scelerisque nisl consectetur et. Donec sed odio dui. Donec ullamcorper nulla non metus\n          auctor fringilla.\n        </p>\n        <p>\n          Cras mattis consectetur purus sit amet fermentum. Cras justo odio, dapibus ac facilisis\n          in, egestas eget quam. Morbi leo risus, porta ac consectetur ac, vestibulum at eros.\n        </p>\n        <p>\n          Praesent commodo cursus magna, vel scelerisque nisl consectetur et. Vivamus sagittis lacus\n          vel augue laoreet rutrum faucibus dolor auctor.\n        </p>\n        <p>\n          Aenean lacinia bibendum nulla sed consectetur. Praesent commodo cursus magna, vel\n          scelerisque nisl consectetur et. Donec sed odio dui. Donec ullamcorper nulla non metus\n          auctor fringilla.\n        </p>\n      </div>\n    </Dialog>\n  );\n\n  return (\n    <div style={{ margin: 20 }}>\n      <p>\n        <button type=\"button\" className=\"btn btn-primary\" onClick={onClick}>\n          show dialog\n        </button>\n        &nbsp;\n        <label>\n          destroy on close:\n          <input type=\"checkbox\" checked={destroyOnClose} onChange={onDestroyOnCloseChange} />\n        </label>\n      </p>\n      {dialog}\n    </div>\n  );\n};\n\nexport default MyControl;\n";
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/examples/draggable.tsx?dumi-raw": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
__mako_require__("docs/examples/draggable.tsx?watch=parent");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _default = "import 'bootstrap/dist/css/bootstrap.css';\nimport * as React from 'react';\nimport Draggable from 'react-draggable';\nimport Dialog from 'rc-dialog';\nimport '../../assets/index.less';\n\nconst MyControl: React.FC = () => {\n  const [visible, setVisible] = React.useState(false);\n  const [disabled, setDisabled] = React.useState(true);\n  const onClick = () => {\n    setVisible(true);\n  };\n  const onClose = () => {\n    setVisible(false);\n  };\n  return (\n    <div style={{ margin: 20 }}>\n      <p>\n        <button type=\"button\" className=\"btn btn-primary\" onClick={onClick}>\n          show dialog\n        </button>\n      </p>\n      <Dialog\n        visible={visible}\n        animation=\"slide-fade\"\n        maskAnimation=\"fade\"\n        onClose={onClose}\n        style={{ width: 600 }}\n        title={\n          <div\n            style={{ width: '100%', cursor: 'pointer' }}\n            onMouseOver={() => {\n              if (disabled) {\n                setDisabled(false);\n              }\n            }}\n            onMouseOut={() => {\n              setDisabled(true);\n            }}\n            // fix eslintjsx-a11y/mouse-events-have-key-events\n            // https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/mouse-events-have-key-events.md\n            onFocus={() => {}}\n            onBlur={() => {}}\n            // end\n          >\n            modal\n          </div>\n        }\n        modalRender={(modal) => <Draggable disabled={disabled}>{modal}</Draggable>}\n      >\n        <div style={{ height: 200 }}>\n          Day before yesterday I saw a rabbit, and yesterday a deer, and today, you.\n        </div>\n      </Dialog>\n    </div>\n  );\n};\n\nexport default MyControl;\n";
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/examples/multiple-Portal.tsx?dumi-raw": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
__mako_require__("docs/examples/multiple-Portal.tsx?watch=parent");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _default = "import * as React from 'react';\nimport Drawer from 'rc-drawer';\nimport 'rc-drawer/assets/index.css';\nimport Dialog from 'rc-dialog';\nimport '../../assets/index.less';\n\nconst Demo: React.FC = () => {\n  const [showDialog, setShowDialog] = React.useState(false);\n  const [showDrawer, setShowDrawer] = React.useState(false);\n\n  const onToggleDrawer = () => {\n    setShowDrawer((value) => !value);\n  };\n\n  const onToggleDialog = () => {\n    setShowDialog((value) => !value);\n  };\n\n  const dialog = (\n    <Dialog\n      visible={showDialog}\n      animation=\"zoom\"\n      maskAnimation=\"fade\"\n      onClose={onToggleDialog}\n      forceRender\n      title=\"basic modal\"\n    >\n      <p>\n        <button type=\"button\" onClick={onToggleDrawer}>\n          show drawer\n        </button>\n      </p>\n      <div style={{ height: 200 }} />\n    </Dialog>\n  );\n  const drawer = (\n    <Drawer open={showDrawer} onClose={onToggleDrawer}>\n      <button type=\"button\" onClick={onToggleDrawer}>\n        close drawer\n      </button>\n    </Drawer>\n  );\n  return (\n    <div>\n      <button type=\"button\" onClick={onToggleDialog}>\n        open dialog\n      </button>\n      <button\n        type=\"button\"\n        onClick={() => {\n          setShowDialog(true);\n          setTimeout(() => {\n            setShowDialog(false);\n          }, 0);\n        }}\n      >\n        quick\n      </button>\n      {dialog}\n      {drawer}\n    </div>\n  );\n};\n\nexport default Demo;\n";
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/examples/pure.tsx?dumi-raw": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "default", {
    enumerable: true,
    get: function() {
        return _default;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
__mako_require__("docs/examples/pure.tsx?watch=parent");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
var _default = "import React from 'react';\nimport { Panel } from 'rc-dialog';\nimport 'rc-select/assets/index.less';\nimport '../../assets/index.less';\n\nconst Demo: React.FC = () => (\n  <Panel prefixCls=\"rc-dialog\" title=\"Title\" closable>\n    Hello World!\n  </Panel>\n);\n\nexport default Demo;\n";
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/index.md?type=demo": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "demos", {
    enumerable: true,
    get: function() {
        return demos;
    }
});
var _interop_require_default = __mako_require__("@swc/helpers/_/_interop_require_default");
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
var _react = /*#__PURE__*/ _interop_require_default._(__mako_require__("node_modules/_react@18.3.1@react/index.js"));
__mako_require__("docs/index.md?watch=parent");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const demos = {};
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
"docs/index.md?type=text": function (module, exports, __mako_require__){
"use strict";
__mako_require__.d(exports, "__esModule", {
    value: true
});
__mako_require__.d(exports, "texts", {
    enumerable: true,
    get: function() {
        return texts;
    }
});
var _interop_require_wildcard = __mako_require__("@swc/helpers/_/_interop_require_wildcard");
var _reactrefresh = /*#__PURE__*/ _interop_require_wildcard._(__mako_require__("node_modules/_react-refresh@0.14.2@react-refresh/runtime.js"));
__mako_require__("docs/index.md?watch=parent");
var prevRefreshReg;
var prevRefreshSig;
prevRefreshReg = self.$RefreshReg$;
prevRefreshSig = self.$RefreshSig$;
self.$RefreshReg$ = (type, id)=>{
    _reactrefresh.register(type, module.id + id);
};
self.$RefreshSig$ = _reactrefresh.createSignatureFunctionForTransform;
const texts = [
    {
        "value": "react dialog component",
        "paraId": 0,
        "tocIndex": 0
    },
    {
        "value": " ",
        "paraId": 1,
        "tocIndex": 0
    },
    {
        "value": " ",
        "paraId": 1,
        "tocIndex": 0
    },
    {
        "value": " ",
        "paraId": 1,
        "tocIndex": 0
    },
    {
        "value": " ",
        "paraId": 1,
        "tocIndex": 0
    },
    {
        "value": " ",
        "paraId": 1,
        "tocIndex": 0
    },
    {
        "value": "http://localhost:8007/examples/",
        "paraId": 2,
        "tocIndex": 2
    },
    {
        "value": "online example: ",
        "paraId": 3,
        "tocIndex": 2
    },
    {
        "value": "https://dialog.react-component.vercel.app/",
        "paraId": 3,
        "tocIndex": 2
    },
    {
        "value": "var Dialog = require('rc-dialog');\n\nReactDOM.render(\n  <Dialog title={title} onClose={callback1} visible>\n      <p>first dialog</p>\n  </Dialog>\n), document.getElementById('t1'));\n\n// use dialog\n",
        "paraId": 4,
        "tocIndex": 4
    },
    {
        "value": "Name",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Type",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Default",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Description",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Version",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "prefixCls",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "String",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "rc-dialog",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "The dialog dom node's prefixCls",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "className",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "String",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "additional className for dialog",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "classNames",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "{ header?: string; body?: string; footer?: string; mask?: string; content?: string; wrapper?: string;  }",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "pass className to target area",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "styles",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "{ header?: CSSProperties; body?: CSSProperties; footer?: CSSProperties; mask?: CSSProperties; content?: CSSProperties; wrapper?: CSSProperties;  }",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "pass styles to target area",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "style",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Object",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "{}",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Root style for dialog element.Such as width, height",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "zIndex",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Number",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "visible",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Boolean",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "false",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "current dialog's visible status",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "animation",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "String",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "part of dialog animation css class name",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "maskAnimation",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "String",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "part of dialog's mask animation css class name",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "transitionName",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "String",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "dialog animation css class name",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "maskTransitionName",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "String",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "mask animation css class name",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "title",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "String|React.Element",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Title of the dialog",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "footer",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "React.Element",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "footer of the dialog",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "closable",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Boolean | ({ closeIcon?: React.ReactNode; disabled?: boolean  } & React.AriaAttributes",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "true",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "whether show close button",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "mask",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Boolean",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "true",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "whether show mask",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "maskClosable",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Boolean",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "true",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "whether click mask to close",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "keyboard",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Boolean",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "true",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "whether support press esc to close",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "mousePosition",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "{x:number,y:number}",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "set pageX and pageY of current mouse(it will cause transform origin to be set).",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "onClose",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "function()",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "called when click close button or mask",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "afterClose",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "function()",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "called when close animation end",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "getContainer",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "function(): HTMLElement",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "to determine where Dialog will be mounted",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "destroyOnClose",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Boolean",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "false",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "to unmount child compenents on onClose",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "closeIcon",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "ReactNode",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "specific the close icon.",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "forceRender",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Boolean",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "false",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Create dialog dom node before dialog first show",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "focusTriggerAfterClose",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Boolean",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "true",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "focus trigger element when dialog closed",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "modalRender",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "(node: ReactNode) => ReactNode",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "Custom modal content render",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "8.3.0",
        "paraId": 5,
        "tocIndex": 6
    },
    {
        "value": "npm install\nnpm start\n",
        "paraId": 6,
        "tocIndex": 7
    },
    {
        "value": "npm test\nnpm run chrome-test\n",
        "paraId": 7,
        "tocIndex": 8
    },
    {
        "value": "npm run coverage\n",
        "paraId": 8,
        "tocIndex": 9
    },
    {
        "value": "open coverage/ dir",
        "paraId": 9,
        "tocIndex": 9
    },
    {
        "value": "rc-dialog is released under the MIT license.",
        "paraId": 10,
        "tocIndex": 10
    }
];
if (prevRefreshReg) self.$RefreshReg$ = prevRefreshReg;
if (prevRefreshSig) self.$RefreshSig$ = prevRefreshSig;
function registerClassComponent(filename, moduleExports) {
    for(const key in moduleExports)try {
        if (key === "__esModule") continue;
        const exportValue = moduleExports[key];
        if (_reactrefresh.isLikelyComponentType(exportValue) && exportValue.prototype && exportValue.prototype.isReactComponent) _reactrefresh.register(exportValue, filename + " " + key);
    } catch (e) {}
}
function $RefreshIsReactComponentLike$(moduleExports) {
    if (_reactrefresh.isLikelyComponentType(moduleExports || moduleExports.default)) return true;
    for(var key in moduleExports)try {
        if (_reactrefresh.isLikelyComponentType(moduleExports[key])) return true;
    } catch (e) {}
    return false;
}
registerClassComponent(module.id, module.exports);
if ($RefreshIsReactComponentLike$(module.exports)) {
    module.meta.hot.accept();
    _reactrefresh.performReactRefresh();
}

},
 }]);
